<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="description" content="This page will walk you through the steps to make your data publicly available so you can meet the requirements for manuscript submission and publication">
<meta name="keywords" content="bioinformaticstranscriptomics,  ">
<title>Depositing RNAseq data to GEO | CHMI services</title>
<link rel="stylesheet" href="css/syntax.css">


<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
<!--<link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">-->
<link rel="stylesheet" href="css/modern-business.css">
<link rel="stylesheet" href="css/lavish-bootstrap.css">
<link rel="stylesheet" href="css/customstyles.css">
<link rel="stylesheet" href="css/theme-blue.css">

<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-cookie/1.4.1/jquery.cookie.min.js"></script>
<script src="js/jquery.navgoco.min.js"></script>


<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/2.0.0/anchor.min.js"></script>
<script src="js/toc.js"></script>
<script src="js/customscripts.js"></script>

<link rel="shortcut icon" href="images/favicon.ico">

<!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
<script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
<![endif]-->

<link rel="alternate" type="application/rss+xml" title="" href="http://localhost:9000/feed.xml">



    <script>
        $(document).ready(function() {
            // Initialize navgoco with default options
            $("#mysidebar").navgoco({
                caretHtml: '',
                accordion: true,
                openClass: 'active', // open
                save: false, // leave false or nav highlighting doesn't work right
                cookie: {
                    name: 'navgoco',
                    expires: false,
                    path: '/'
                },
                slide: {
                    duration: 400,
                    easing: 'swing'
                }
            });

            $("#collapseAll").click(function(e) {
                e.preventDefault();
                $("#mysidebar").navgoco('toggle', false);
            });

            $("#expandAll").click(function(e) {
                e.preventDefault();
                $("#mysidebar").navgoco('toggle', true);
            });

        });

    </script>
    <script>
        $(function () {
            $('[data-toggle="tooltip"]').tooltip()
        })
    </script>
    <script>
        $(document).ready(function() {
            $("#tg-sb-link").click(function() {
                $("#tg-sb-sidebar").toggle();
                $("#tg-sb-content").toggleClass('col-md-9');
                $("#tg-sb-content").toggleClass('col-md-12');
                $("#tg-sb-icon").toggleClass('fa-toggle-on');
                $("#tg-sb-icon").toggleClass('fa-toggle-off');
            });
        });
    </script>
    

</head>
<body>
<!-- Navigation -->
<nav class="navbar navbar-inverse navbar-fixed-top">
    <div class="container topnavlinks">
        <div class="navbar-header">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a class="fa fa-home fa-lg navbar-brand" href="index.html">&nbsp;<span class="projectTitle"> Center for Host-Microbial Interactions</span></a>
        </div>
        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
            <ul class="nav navbar-nav navbar-right">
                <!-- toggle sidebar button -->
                <li><a id="tg-sb-link" href="#"><i id="tg-sb-icon" class="fa fa-toggle-on"></i> Nav</a></li>
                <!-- entries without drop-downs appear here -->
                
                
                
                <!-- entries with drop-downs appear here -->
                <!-- conditional logic to control which topnav appears for the audience defined in the configuration file.-->
                
                
                
                
			<li>



  <a class="email" title="Submit feedback" href="#" onclick="javascript:window.location='mailto:beiting@upenn.edu?subject=CHMI services feedback&body=I have some feedback about the Depositing RNAseq data to GEO page: ' + window.location.href;"><i class="fa fa-envelope-o"></i> Feedback</a>

</li>

		
                <!--comment out this block if you want to hide search-->
                <li>
                    <!--start search-->
                    <div id="search-demo-container">
                        <input type="text" id="search-input" placeholder="search...">
                        <ul id="results-container"></ul>
                    </div>
                    <script src="js/jekyll-search.js" type="text/javascript"></script>
                    <script type="text/javascript">
                            SimpleJekyllSearch.init({
                                searchInput: document.getElementById('search-input'),
                                resultsContainer: document.getElementById('results-container'),
                                dataSource: 'search.json',
                                searchResultTemplate: '<li><a href="{url}" title="Depositing RNAseq data to GEO">{title}</a></li>',
                    noResultsText: 'No results found.',
                            limit: 10,
                            fuzzy: true,
                    })
                    </script>
                    <!--end search-->
                </li>
            </ul>
        </div>
        </div>
        <!-- /.container -->
</nav>

<!-- Page Content -->
<div class="container">
    <div class="col-lg-12">&nbsp;</div>
    <!-- Content Row -->
    <div class="row">
        
        
            <!-- Sidebar Column -->
            <div class="col-md-3" id="tg-sb-sidebar">
                












<ul id="mysidebar" class="nav">
    <li class="sidebarTitle">CHMI services </li>
    
    
    
        
    
    <li>
        <a href="#">Overview</a>
        <ul>
            
            
            
            <li><a href="index.html">Welcome</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Bulk RNAseq</a>
        <ul>
            
            
            
            <li><a href="mydoc_RNAseq_FAQs.html">FAQs</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_TruSeq.html">mRNA library prep</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_total_TruSeq.html">Total transcriptome library prep</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_lowInput_RNAseq.html">Low input protocol</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_FFPE_RNAseq.html">FFPE-seq</a></li>
            
            
            
            <li class="subfolders">
                <a href="#">RNAseq data analysis</a>
                <ul>
                    
                    
                    
                    <li><a href="mydoc_RNAseq_QA.html">Project management and QA</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_kallisto.html">Read mapping with Kallisto</a></li>
                    
                    
                    
                    
                    
                    <li class="active"><a href="mydoc_RNAseq_depositToGEO.html">Submitting data to GEO</a></li>
                    
                    
                    
                </ul>
            </li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Single cell genomics</a>
        <ul>
            
            
            
            <li><a href="mydoc_scRNAseq_FAQs.html">scRNAseq FAQs</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_10X_prep.html">10X sample prep</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_inDrop_encapsulation.html">inDrop encapsulation</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_inDrop_libraryPrep.html">inDrop library prep</a></li>
            
            
            
            <li class="subfolders">
                <a href="#">scRNAseq data analysis</a>
                <ul>
                    
                    
                    
                    <li><a href="mydoc_CellRanger.html">10X CellRanger</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_CellRanger.html">BUStools</a></li>
                    
                    
                    
                </ul>
            </li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Microbiology</a>
        <ul>
            
            
            
            <li><a href="mydoc_micro_FAQs.html">Microbiology FAQs</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_16S.html">16S rRNA gene sequencing</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_NexteraXT.html">Shotgun metagenomics</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_16S_Sanger.html">Sanger sequencing of 16S</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_mock.html">Culture collection</a></li>
            
            
            
            <li class="subfolders">
                <a href="#">Data analysis</a>
                <ul>
                    
                    
                    
                    <li><a href="mydoc_qiime2.html">QIIME2</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_microbiomeDB_deposit.html">microbiomeDB.org</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_sourmash.html">Sourmash</a></li>
                    
                    
                    
                </ul>
            </li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Compute resources</a>
        <ul>
            
            
            
            <li><a href="mydoc_linux.html">Linux cluster</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_software.html">Available software</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_RStudio.html">R Studio Server</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Cell culture</a>
        <ul>
            
            
            
            <li><a href="mydoc_3Dculture.html">3D intestinal cultures</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_3Dculture_microscopy.html">Immunofluorescence for 3D cultures</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Portable sequencing</a>
        <ul>
            
            
            
            <li><a href="mydoc_biomeme.html">ONT MinION set-up</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Portable QPCR</a>
        <ul>
            
            
            
            <li><a href="mydoc_biomeme.html">Biomeme setup</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_biomeme_avian.html">Avian</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_biomeme_human.html">Cutaneous leishmaniasis</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Workshops</a>
        <ul>
            
            
            
            <li><a href="mydoc_cloudMeta.html">Metagenomics in the cloud</a></li>
            
            
            
            
        </ul>
     </li>
       
        
        
        <!-- if you aren't using the accordion, uncomment this block:
           <p class="external">
               <a href="#" id="collapseAll">Collapse All</a> | <a href="#" id="expandAll">Expand All</a>
           </p>
           -->
</ul>

<!-- this highlights the active parent class in the navgoco sidebar. this is critical so that the parent expands when you're viewing a page. This must appear below the sidebar code above. Otherwise, if placed inside customscripts.js, the script runs before the sidebar code runs and the class never gets inserted.-->
<script>$("li.active").parents('li').toggleClass("active");</script>

            </div>
            
        

        <!-- Content Column -->
        <div class="col-md-9" id="tg-sb-content">
            <div class="post-header">
   <h1 class="post-title-main">Depositing RNAseq data to GEO</h1>
</div>



<div class="post-content">

   
    <div class="summary">This page will walk you through the steps to make your data publicly available so you can meet the requirements for manuscript submission and publication</div>
   

    
    
<!-- this handles the automatic toc. use ## for subheads to auto-generate the on-page minitoc. if you use html tags, you must supply an ID for the heading element in order for it to appear in the minitoc. -->
<script>
$( document ).ready(function() {
  // Handler for .ready() called.

$('#toc').toc({ minimumHeaders: 0, listType: 'ul', showSpeed: 0, headers: 'h2,h3,h4' });

/* this offset helps account for the space taken up by the floating toolbar. */
$('#toc').on('click', 'a', function() {
  var target = $(this.getAttribute('href'))
    , scroll_target = target.offset().top

  $(window).scrollTop(scroll_target - 10);
  return false
})
  
});
</script>

<div id="toc"></div>

    


    

    <a target="_blank" href="https://github.com/CHMI-SOPs/CHMI-SOPs.github.io/tree/master/pages/mydoc/mydoc_RNAseq_depositToGEO.md" class="btn btn-default githubEditButton" role="button"><i class="fa fa-github fa-lg"></i> Edit me</a>

    

  <h2 id="intro">Intro</h2>
<p>So you’re ready to submit a manuscript describing your hard work…congratulations!  If you’re reading this page, then chances are you have RNAseq data that you want to post to an approved data repository so that you can get an accession number to include in your manuscript submission.  This is the first step in ensuring that the broader research community can access your raw and processed data from any sequencing experiment(s) carried out in your publication.  The <a href="https://www.ncbi.nlm.nih.gov/sra">Sequence Read Archive</a> (SRA) is the defacto home for all sequencing related studies, but can be confusing to navigate.  Thankfully, the <a href="https://www.ncbi.nlm.nih.gov/geo/">Gene Expression Omnibus</a>, will accept deposits of gene expression profiling data from RNAseq experiments and will pass this data onto to the SRA on your behalf.  This is definitely the way to go, since the process of submitting data to GEO is pretty straightforward.  The steps below will walk you through this process.  Be aware that GEO’s own instructions are available <a href="https://www.ncbi.nlm.nih.gov/geo/info/seq.html">here</a>, but our instructions are simple.</p>

<h2 id="what-youll-need">What you’ll need</h2>

<ul>
  <li>an account on myNCBI.  If you don’t already have one, signup <a href="https://www.ncbi.nlm.nih.gov/account/">here</a></li>
  <li>the free FTP client, <a href="https://filezilla-project.org/">FileZilla</a></li>
  <li>access to your raw .fastq files from your sequencing experiment. Keep each fastq file in the compressed (.gzip) form at all times.</li>
  <li>access to a table of ‘processed’ gene expression data.  This will need to be either raw counts, or normalized data produced by DESeq2 or EdgeR (or both files).</li>
</ul>

<h2 id="step-1-complete-metadata-spreadsheet">Step 1: Complete metadata spreadsheet</h2>

<p>This is most time consuming part of whole process, but also the most important.  <a href="http://CHMI-sops.github.io/papers/GEOsubmission.xlsx">Download this example excel file</a> from one of our own RNAseq studies, and replace with the appropriate information from your own experiment.  This document should describe each sample, how it was handled, and how your samples relate to the raw fastq files.  The more ‘metadata’ you can include here, the more useful you make this data to outside investigators, so don’t skimp on the details.</p>

<p>To complete this form you will need to generate a unique checksum for each file that you intend to transfer to GEO (both fastq files, as well as an process data files).  Checksums confirm that the file is not corrupt.  They are easy to generate, but do require that you open a command line program (e.g. Terminal on a Mac).  Once you have a terminal window open, navigate to the directory on your computer where you have all the files you plan to transfer.</p>

<p>Use the <code class="highlighter-rouge">md5</code> command followed by the name of your file to generate the 32 character checksum.  You’ll want to copy and paste this into your excel submission form.
Here’s an example of what this looks like:</p>

<div class="highlighter-rouge"><pre class="highlight"><code>md5 GEOsubmission.xlsx 
MD5 (GEOsubmission.xlsx) = 39eae0dbcc54367c588d2815876747f4
</code></pre>
</div>

<h2 id="step-2-preparing-to-transfer">Step 2: Preparing to transfer</h2>

<ol>
  <li>Create a folder anywhere on your computer and give it the same name as your myNCBI username (for me this is ‘danielbeiting’).</li>
  <li>Transfer all your raw and processed files, and your metadata spreadsheet completed in step 1, to this folder.</li>
  <li>If the total size of this folder is &gt;1Tb, you will need to <a href="geo@ncbi.nlm.nih.gov">email GEO</a> with a list of your file names and their checksums, so that they can manage server space accordingly.</li>
</ol>

<h2 id="step-3-transfer-your-data">Step 3: Transfer your data</h2>

<ol>
  <li>Launch FileZilla and connect to the GEO storage server using the following credentials:</li>
</ol>

<ul>
  <li>Host: ftp-private.ncbi.nlm.nih.gov</li>
  <li>username: geo</li>
  <li>password: (this will specific to your myNCBI account). To get your user-specific password, make sure you’re logged into your myNCBI account on the web, and then <strong><a href="https://www.ncbi.nlm.nih.gov/geo/info/submissionftp.html">go to this webpage</a></strong>.  Scroll down and you see your login credentials.  You’ll need to enter this info into FileZilla to connect.</li>
</ul>

<ol>
  <li>
    <p>Once connected via FileZilla, simply drag your folder from your local computer to the GEO server</p>
  </li>
  <li>
    <p>Now you must give the crew at GEO a heads-up that you’ve submitted data to their server.  <strong>Don’t skip this step or they will delete your submission.</strong>. <a href="geo@ncbi.nlm.nih.gov">Send an email to GEO</a>, and include the following information in the email:</p>
  </li>
</ol>

<ul>
  <li>Your GEO account username.  If depositing to my account, use ‘danielbeiting’</li>
  <li>Names of the directory and files deposited</li>
  <li>Choose a date for public release (this can be immediate, or you can choose up to a maximum of 3yrs.  You can always extend later)</li>
</ul>

<ol>
  <li>Congratulations, you’re done!  You will get an email from GEO with your accession number.  Include this number in your manuscript.  If the data is kept private on GEO, but reviewers request a link to access the data (they are entitled to this), you can get a reviewer link through your account on GEO.  If there are any problems with your submission, GEO will contant you and you’ll get some instructions on how to resolve the issue before the submission is accepted.</li>
</ol>



    <div class="tags">
        
        <b>Tags: </b>
        
        
        
        <a href="tag_bioinformatics.html" class="btn btn-default navbar-btn cursorNorm" role="button">bioinformatics</a>
        
        
        
        <a href="tag_transcriptomics.html" class="btn btn-default navbar-btn cursorNorm" role="button">transcriptomics</a>
        
        
        
    </div>



    <div id="disqus_thread"></div>
    <script type="text/javascript">
        /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
        var disqus_shortname = 'http-hostmicrobe-org'; // required: replace example with your forum shortname

        /* * * DON'T EDIT BELOW THIS LINE * * */
        (function() {
            var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
            dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
            (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
        })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>


</div>

<hr class="shaded"/>

<footer>
            <div class="row">
                <div class="col-lg-12 footer">
               &copy;2020 PennVet Center for Host-Microbial Interactions. All rights reserved. <br />
 Site last generated: Mar 23, 2020 <br />
<p><img src="images/CHMI_logo.png" alt="Company logo"/></p>
                </div>
            </div>
</footer>


        </div>
    <!-- /.row -->
</div>
<!-- /.container -->
    </div>

</body>

<!-- the google_analytics_id gets auto inserted from the config file -->



<script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','//www.google-analytics.com/analytics.js','ga');ga('create','UA-131349832-1','auto');ga('require','displayfeatures');ga('send','pageview');</script>


</html>
