<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="description" content="Cell Ranger is the command-line software for preprocessing raw sequence data from a 10X single cell sequencing experiment.  The output from Cell Ranger os a ...">
<meta name="keywords" content="bioinformaticssingle_cell_genomics,  ">
<title>Cell Ranger | CHMI services</title>
<link rel="stylesheet" href="css/syntax.css">


<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
<!--<link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">-->
<link rel="stylesheet" href="css/modern-business.css">
<link rel="stylesheet" href="css/lavish-bootstrap.css">
<link rel="stylesheet" href="css/customstyles.css">
<link rel="stylesheet" href="css/theme-blue.css">

<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-cookie/1.4.1/jquery.cookie.min.js"></script>
<script src="js/jquery.navgoco.min.js"></script>


<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/2.0.0/anchor.min.js"></script>
<script src="js/toc.js"></script>
<script src="js/customscripts.js"></script>

<link rel="shortcut icon" href="images/favicon.ico">

<!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
<script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
<![endif]-->

<link rel="alternate" type="application/rss+xml" title="" href="http://localhost:9000/feed.xml">



    <script>
        $(document).ready(function() {
            // Initialize navgoco with default options
            $("#mysidebar").navgoco({
                caretHtml: '',
                accordion: true,
                openClass: 'active', // open
                save: false, // leave false or nav highlighting doesn't work right
                cookie: {
                    name: 'navgoco',
                    expires: false,
                    path: '/'
                },
                slide: {
                    duration: 400,
                    easing: 'swing'
                }
            });

            $("#collapseAll").click(function(e) {
                e.preventDefault();
                $("#mysidebar").navgoco('toggle', false);
            });

            $("#expandAll").click(function(e) {
                e.preventDefault();
                $("#mysidebar").navgoco('toggle', true);
            });

        });

    </script>
    <script>
        $(function () {
            $('[data-toggle="tooltip"]').tooltip()
        })
    </script>
    <script>
        $(document).ready(function() {
            $("#tg-sb-link").click(function() {
                $("#tg-sb-sidebar").toggle();
                $("#tg-sb-content").toggleClass('col-md-9');
                $("#tg-sb-content").toggleClass('col-md-12');
                $("#tg-sb-icon").toggleClass('fa-toggle-on');
                $("#tg-sb-icon").toggleClass('fa-toggle-off');
            });
        });
    </script>
    

</head>
<body>
<!-- Navigation -->
<nav class="navbar navbar-inverse navbar-fixed-top">
    <div class="container topnavlinks">
        <div class="navbar-header">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a class="fa fa-home fa-lg navbar-brand" href="index.html">&nbsp;<span class="projectTitle"> Center for Host-Microbial Interactions</span></a>
        </div>
        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
            <ul class="nav navbar-nav navbar-right">
                <!-- toggle sidebar button -->
                <li><a id="tg-sb-link" href="#"><i id="tg-sb-icon" class="fa fa-toggle-on"></i> Nav</a></li>
                <!-- entries without drop-downs appear here -->
                
                
                
                <!-- entries with drop-downs appear here -->
                <!-- conditional logic to control which topnav appears for the audience defined in the configuration file.-->
                
                
                
                
			<li>



  <a class="email" title="Submit feedback" href="#" onclick="javascript:window.location='mailto:beiting@upenn.edu?subject=CHMI services feedback&body=I have some feedback about the Cell Ranger page: ' + window.location.href;"><i class="fa fa-envelope-o"></i> Feedback</a>

</li>

		
                <!--comment out this block if you want to hide search-->
                <li>
                    <!--start search-->
                    <div id="search-demo-container">
                        <input type="text" id="search-input" placeholder="search...">
                        <ul id="results-container"></ul>
                    </div>
                    <script src="js/jekyll-search.js" type="text/javascript"></script>
                    <script type="text/javascript">
                            SimpleJekyllSearch.init({
                                searchInput: document.getElementById('search-input'),
                                resultsContainer: document.getElementById('results-container'),
                                dataSource: 'search.json',
                                searchResultTemplate: '<li><a href="{url}" title="Cell Ranger">{title}</a></li>',
                    noResultsText: 'No results found.',
                            limit: 10,
                            fuzzy: true,
                    })
                    </script>
                    <!--end search-->
                </li>
            </ul>
        </div>
        </div>
        <!-- /.container -->
</nav>

<!-- Page Content -->
<div class="container">
    <div class="col-lg-12">&nbsp;</div>
    <!-- Content Row -->
    <div class="row">
        
        
            <!-- Sidebar Column -->
            <div class="col-md-3" id="tg-sb-sidebar">
                












<ul id="mysidebar" class="nav">
    <li class="sidebarTitle">CHMI services </li>
    
    
    
        
    
    <li>
        <a href="#">Overview</a>
        <ul>
            
            
            
            <li><a href="index.html">Welcome</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Bulk RNAseq</a>
        <ul>
            
            
            
            <li><a href="mydoc_RNAseq_FAQs.html">FAQs</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_TruSeq.html">mRNA library prep</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_total_TruSeq.html">Total transcriptome library prep</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_lowInput_RNAseq.html">Low input protocol</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_FFPE_RNAseq.html">FFPE-seq</a></li>
            
            
            
            <li class="subfolders">
                <a href="#">RNAseq data analysis</a>
                <ul>
                    
                    
                    
                    <li><a href="mydoc_RNAseq_QA.html">Project management and QA</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_kallisto.html">Read mapping with Kallisto</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_RNAseq_depositToGEO.html">Submitting data to GEO</a></li>
                    
                    
                    
                </ul>
            </li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Single cell genomics</a>
        <ul>
            
            
            
            <li><a href="mydoc_scRNAseq_FAQs.html">scRNAseq FAQs</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_10X_prep.html">10X sample prep</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_inDrop_encapsulation.html">inDrop encapsulation</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_inDrop_libraryPrep.html">inDrop library prep</a></li>
            
            
            
            <li class="subfolders">
                <a href="#">scRNAseq data analysis</a>
                <ul>
                    
                    
                    
                    <li class="active"><a href="mydoc_CellRanger.html">10X CellRanger</a></li>
                    
                    
                    
                    
                    
                    <li class="active"><a href="mydoc_CellRanger.html">BUStools</a></li>
                    
                    
                    
                </ul>
            </li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Microbiology</a>
        <ul>
            
            
            
            <li><a href="mydoc_micro_FAQs.html">Microbiology FAQs</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_16S.html">16S rRNA gene sequencing</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_NexteraXT.html">Shotgun metagenomics</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_16S_Sanger.html">Sanger sequencing of 16S</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_mock.html">Culture collection</a></li>
            
            
            
            <li class="subfolders">
                <a href="#">Data analysis</a>
                <ul>
                    
                    
                    
                    <li><a href="mydoc_qiime2.html">QIIME2</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_microbiomeDB_deposit.html">microbiomeDB.org</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_sourmash.html">Sourmash</a></li>
                    
                    
                    
                </ul>
            </li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Compute resources</a>
        <ul>
            
            
            
            <li><a href="mydoc_linux.html">Linux cluster</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_software.html">Available software</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_RStudio.html">R Studio Server</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Cell culture</a>
        <ul>
            
            
            
            <li><a href="mydoc_3Dculture.html">3D intestinal cultures</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_3Dculture_microscopy.html">Immunofluorescence for 3D cultures</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Portable sequencing</a>
        <ul>
            
            
            
            <li><a href="mydoc_biomeme.html">ONT MinION set-up</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Portable QPCR</a>
        <ul>
            
            
            
            <li><a href="mydoc_biomeme.html">Biomeme setup</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_biomeme_avian.html">Avian</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_biomeme_human.html">Cutaneous leishmaniasis</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Workshops</a>
        <ul>
            
            
            
            <li><a href="mydoc_cloudMeta.html">Metagenomics in the cloud</a></li>
            
            
            
            
        </ul>
     </li>
       
        
        
        <!-- if you aren't using the accordion, uncomment this block:
           <p class="external">
               <a href="#" id="collapseAll">Collapse All</a> | <a href="#" id="expandAll">Expand All</a>
           </p>
           -->
</ul>

<!-- this highlights the active parent class in the navgoco sidebar. this is critical so that the parent expands when you're viewing a page. This must appear below the sidebar code above. Otherwise, if placed inside customscripts.js, the script runs before the sidebar code runs and the class never gets inserted.-->
<script>$("li.active").parents('li').toggleClass("active");</script>

            </div>
            
        

        <!-- Content Column -->
        <div class="col-md-9" id="tg-sb-content">
            <div class="post-header">
   <h1 class="post-title-main">Cell Ranger</h1>
</div>



<div class="post-content">

   
    <div class="summary">Cell Ranger is the command-line software for preprocessing raw sequence data from a 10X single cell sequencing experiment.  The output from Cell Ranger os a count matrix where rows are genes and columns are individual cells.</div>
   

    
    
<!-- this handles the automatic toc. use ## for subheads to auto-generate the on-page minitoc. if you use html tags, you must supply an ID for the heading element in order for it to appear in the minitoc. -->
<script>
$( document ).ready(function() {
  // Handler for .ready() called.

$('#toc').toc({ minimumHeaders: 0, listType: 'ul', showSpeed: 0, headers: 'h2,h3,h4' });

/* this offset helps account for the space taken up by the floating toolbar. */
$('#toc').on('click', 'a', function() {
  var target = $(this.getAttribute('href'))
    , scroll_target = target.offset().top

  $(window).scrollTop(scroll_target - 10);
  return false
})
  
});
</script>

<div id="toc"></div>

    


    

    <a target="_blank" href="https://github.com/CHMI-SOPs/CHMI-SOPs.github.io/tree/master/pages/mydoc/mydoc_CellRanger.md" class="btn btn-default githubEditButton" role="button"><i class="fa fa-github fa-lg"></i> Edit me</a>

    

  <h2 id="intro">Intro</h2>

<h2 id="before-starting">Before starting</h2>

<p>The instructions below assume the following:</p>

<ul>
  <li>You have Cell Ranger installed on a computer with <a href="https://support.10xgenomics.com/single-cell-atac/software/overview/system-requirements">the recommended compute resources</a>.</li>
  <li>Cell Ranger is in your <a href="https://opensource.com/article/17/6/set-path-linux">$PATH</a>.</li>
  <li>You have a .csv file (e.g. in Excel) that identifies samples with barcodes</li>
  <li>You have access to the raw .bcls file from a single cell sequencing experiment.</li>
  <li>You have some other basic linux software installed on the same computer (also in your $PATH), including <a href="https://www.gnu.org/software/wget/">wget</a> for downloading files.</li>
  <li>You have organized your working directory as <code class="highlighter-rouge">/data/userName/experimentName/runName/</code></li>
</ul>

<h2 id="prepare-your-reference-genome-file">Prepare your reference genome file</h2>

<h3 id="mouse-and-human">mouse and human</h3>

<p>If your cells came from mice or humans, then you’re in luck!  You can just download the reference file from <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/downloads/latest">here</a>.  Alternatively, if you’re working on our linux machine, then you can find these references and more in <code class="highlighter-rouge">/data/reference_db/10X</code></p>

<h3 id="other-organisms">other organisms</h3>

<p>If your cells are <strong>not</strong> mouse or human, then you’ll need to do a bit of work to get your reference file set-up.</p>

<p>Begin by downloading the reference genome fasta file and .gtf file for your organism of interest.  You can browse available files at the <a href="https://useast.ensembl.org/info/data/ftp/index.html">Ensemble FTP site</a>.</p>

<p>Once you’ve identified the fasta (.fa) and .gtf files you need, download them using wget</p>

<div class="language-shell highlighter-rouge"><pre class="highlight"><code><span class="c"># using rat as an example</span>
<span class="c"># first the genome fasta file</span>
wget ftp://ftp.ensembl.org/pub/release-99/fasta/rattus_norvegicus/dna/Rattus_norvegicus.Rnor_6.0.dna.toplevel.fa.gz
<span class="c"># then the .gtf file</span>
wget ftp://ftp.ensembl.org/pub/release-99/gtf/rattus_norvegicus/Rattus_norvegicus.Rnor_6.0.99.chr.gtf.gz
</code></pre>
</div>

<p>Unzip both files using gunzip</p>

<div class="language-shell highlighter-rouge"><pre class="highlight"><code>gunzip Rattus_norvegicus.Rnor_6.0.99.chr.gtf.gz <span class="c"># gtf file from Ensembl</span>
gunzip Rattus_norvegicus.Rnor_6.0.dna.toplevel.fa.gz <span class="c"># genome fasta from Ensembl</span>
</code></pre>
</div>

<p>Use Cell Ranger’s <strong><a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/advanced/references#mkgtf">mkgtf</a></strong> function to filter your original GTF file to leave only those genes that are annotated as protein coding.</p>

<div class="language-shell highlighter-rouge"><pre class="highlight"><code>cellranger mkgtf <span class="se">\</span>
Rattus_norvegicus.Rnor_6.0.99.chr.gtf <span class="se">\ </span><span class="c"># gtf file from Ensembl</span>
Rattus_norvegicus.Rnor_6.0.99.chr.filtered.gtf <span class="se">\ </span><span class="c"># output name for filtered gtf</span>
--attribute<span class="o">=</span>gene_biotype:protein_coding <span class="c"># filtering on protein coding genes</span>
</code></pre>
</div>

<p>Use Cell Ranger’s <strong><a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/advanced/references#mkref">mkref</a></strong> function to build a index for read alignment.  This uses the splice-aware aligner, <a href="https://github.com/alexdobin/STAR">STAR</a> to accomplish this.</p>

<div class="language-shell highlighter-rouge"><pre class="highlight"><code>cellranger mkref <span class="se">\</span>
--genome<span class="o">=</span>Rattus_norvegicus_genome_forRNA <span class="se">\ </span><span class="c">#output folder name</span>
--fasta<span class="o">=</span>Rattus_norvegicus.Rnor_6.0.dna.toplevel.fa <span class="se">\ </span><span class="c"># reference fasta file </span>
--genes<span class="o">=</span>Rattus_norvegicus.Rnor_6.0.99.chr.filtered.gtf <span class="se">\ </span><span class="c"># filtered gtf file from above</span>
--nthreads<span class="o">=</span>24 <span class="c"># number of threads to use (set to 24 on our linux machine)</span>
</code></pre>
</div>

<h2 id="prepare-fastq-files">Prepare fastq files</h2>

<p>#go to the correct folder
cd /data/userName/experimentName</p>

<p>Use Cell Ranger’s <strong><a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/mkfastq">mkfastq</a></strong> function to convert the bcl files from your illumina sequencing run to fastq files.  mkfastq is basically a wrapper around Illumina’s bcl2fastq program, but with a few extra features, namely that it handles 10X indices for demultiplexing samples, and generates some quality control metrics that are specific to the 10X platform.</p>

<div class="language-shell highlighter-rouge"><pre class="highlight"><code>cellranger mkfastq --id<span class="o">=</span>fastqGroup <span class="se">\ </span><span class="c"># name you're giving to the fastq group</span>
                   --run<span class="o">=</span>/data/userName/BCL <span class="se">\ </span><span class="c"># the folder with your BCL files</span>
                   --csv<span class="o">=</span>/data/userName/experimentName/demux.csv <span class="c"># csv file that contains the barcode assignments and sample names.</span>
</code></pre>
</div>

<h2 id="generate-counts">Generate counts</h2>

<h3 id="process-one-sample-at-a-time">process one sample at a time</h3>

<p>Use Cell Ranger’s <strong><a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/tutorial_ct">count</a></strong> function aligns sequencing reads in FASTQ files to your reference transcriptome and generates a .cloupe file for visualization and analysis in Loupe Browser, along with a number of other outputs compatible with other publicly-available tools for further analysis.</p>

<div class="language-shell highlighter-rouge"><pre class="highlight"><code>cellranger count --id<span class="o">=</span>outputName <span class="se">\ </span><span class="c"># name for the output</span>
                 --fastqs<span class="o">=</span>/data/userName/experimentName/fastqGroup/outs/fastq_path/serialNumber/ <span class="se">\ </span><span class="c"># path to the fastq files, which should end in the serial number for the flow cell used. Be aware which folder you are in when you run this command!</span>
                 --sample<span class="o">=</span>sampleName <span class="se">\ </span><span class="c"># name of the sample to be processed. must match the name you gave in your csv file!</span>
                 --transcriptome<span class="o">=</span>/data/reference_db/10X/refdata-cellranger-mm10-3.0.0 <span class="c"># path to your transcriptome created with mkref above.  This example uses mouse</span>
</code></pre>
</div>

<h3 id="process-multiple-samples">process multiple samples</h3>

<p>This is useful if you have multiple samples, or if you used <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/feature-bc">feature barcoding</a> to multiplex multiple samples in a single well of a 10X chip.</p>

<div class="alert alert-info" role="alert"><i class="fa fa-info-circle"></i> <b>Note:</b> There are no <code class="highlighter-rouge">sample</code> or <code class="highlighter-rouge">fastqs</code> arguments when processing multiple samples, instead, there are two .csv files - one for libraries and a second for your feature reference.</div>

<div class="language-shell highlighter-rouge"><pre class="highlight"><code>cellranger count --id<span class="o">=</span>Count_output_combined2 <span class="se">\</span>
                 --libraries<span class="o">=</span>/data/userName/libraries.csv <span class="se">\</span>
                 --feature-ref<span class="o">=</span>/data/userName/featurereference.csv <span class="se">\</span>
                 --transcriptome<span class="o">=</span>/data/reference_db/10X/refdata-cellranger-mm10-3.0.0
</code></pre>
</div>



    <div class="tags">
        
        <b>Tags: </b>
        
        
        
        <a href="tag_bioinformatics.html" class="btn btn-default navbar-btn cursorNorm" role="button">bioinformatics</a>
        
        
        
        <a href="tag_single_cell_genomics.html" class="btn btn-default navbar-btn cursorNorm" role="button">single_cell_genomics</a>
        
        
        
    </div>



    <div id="disqus_thread"></div>
    <script type="text/javascript">
        /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
        var disqus_shortname = 'http-hostmicrobe-org'; // required: replace example with your forum shortname

        /* * * DON'T EDIT BELOW THIS LINE * * */
        (function() {
            var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
            dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
            (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
        })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>


</div>

<hr class="shaded"/>

<footer>
            <div class="row">
                <div class="col-lg-12 footer">
               &copy;2020 PennVet Center for Host-Microbial Interactions. All rights reserved. <br />
 Site last generated: Mar 23, 2020 <br />
<p><img src="images/CHMI_logo.png" alt="Company logo"/></p>
                </div>
            </div>
</footer>


        </div>
    <!-- /.row -->
</div>
<!-- /.container -->
    </div>

</body>

<!-- the google_analytics_id gets auto inserted from the config file -->



<script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','//www.google-analytics.com/analytics.js','ga');ga('create','UA-131349832-1','auto');ga('require','displayfeatures');ga('send','pageview');</script>


</html>
