<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="description" content="Identification of unknown bacterial isolates using Sanger sequencing of the 16S rRNA gene">
<meta name="keywords" content="microbiome,  ">
<title>Identification of unknown bacterial isolates using Sanger sequencing of the 16S rRNA gene | CHMI services</title>
<link rel="stylesheet" href="css/syntax.css">


<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
<!--<link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">-->
<link rel="stylesheet" href="css/modern-business.css">
<link rel="stylesheet" href="css/lavish-bootstrap.css">
<link rel="stylesheet" href="css/customstyles.css">
<link rel="stylesheet" href="css/theme-blue.css">

<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-cookie/1.4.1/jquery.cookie.min.js"></script>
<script src="js/jquery.navgoco.min.js"></script>


<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/2.0.0/anchor.min.js"></script>
<script src="js/toc.js"></script>
<script src="js/customscripts.js"></script>

<link rel="shortcut icon" href="images/favicon.ico">

<!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
<script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
<![endif]-->

<link rel="alternate" type="application/rss+xml" title="" href="http://localhost:4000/feed.xml">



    <script>
        $(document).ready(function() {
            // Initialize navgoco with default options
            $("#mysidebar").navgoco({
                caretHtml: '',
                accordion: true,
                openClass: 'active', // open
                save: false, // leave false or nav highlighting doesn't work right
                cookie: {
                    name: 'navgoco',
                    expires: false,
                    path: '/'
                },
                slide: {
                    duration: 400,
                    easing: 'swing'
                }
            });

            $("#collapseAll").click(function(e) {
                e.preventDefault();
                $("#mysidebar").navgoco('toggle', false);
            });

            $("#expandAll").click(function(e) {
                e.preventDefault();
                $("#mysidebar").navgoco('toggle', true);
            });

        });

    </script>
    <script>
        $(function () {
            $('[data-toggle="tooltip"]').tooltip()
        })
    </script>
    <script>
        $(document).ready(function() {
            $("#tg-sb-link").click(function() {
                $("#tg-sb-sidebar").toggle();
                $("#tg-sb-content").toggleClass('col-md-9');
                $("#tg-sb-content").toggleClass('col-md-12');
                $("#tg-sb-icon").toggleClass('fa-toggle-on');
                $("#tg-sb-icon").toggleClass('fa-toggle-off');
            });
        });
    </script>
    

</head>
<body>
<!-- Navigation -->
<nav class="navbar navbar-inverse navbar-fixed-top">
    <div class="container topnavlinks">
        <div class="navbar-header">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a class="fa fa-home fa-lg navbar-brand" href="index.html">&nbsp;<span class="projectTitle"> Center for Host-Microbial Interactions</span></a>
        </div>
        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
            <ul class="nav navbar-nav navbar-right">
                <!-- toggle sidebar button -->
                <li><a id="tg-sb-link" href="#"><i id="tg-sb-icon" class="fa fa-toggle-on"></i> Nav</a></li>
                <!-- entries without drop-downs appear here -->
                
                
                
                <!-- entries with drop-downs appear here -->
                <!-- conditional logic to control which topnav appears for the audience defined in the configuration file.-->
                
                
                
                
			<li>



  <a class="email" title="Submit feedback" href="#" onclick="javascript:window.location='mailto:beiting@upenn.edu?subject=CHMI services feedback&body=I have some feedback about the Identification of unknown bacterial isolates using Sanger sequencing of the 16S rRNA gene page: ' + window.location.href;"><i class="fa fa-envelope-o"></i> Feedback</a>

</li>

		
                <!--comment out this block if you want to hide search-->
                <li>
                    <!--start search-->
                    <div id="search-demo-container">
                        <input type="text" id="search-input" placeholder="search...">
                        <ul id="results-container"></ul>
                    </div>
                    <script src="js/jekyll-search.js" type="text/javascript"></script>
                    <script type="text/javascript">
                            SimpleJekyllSearch.init({
                                searchInput: document.getElementById('search-input'),
                                resultsContainer: document.getElementById('results-container'),
                                dataSource: 'search.json',
                                searchResultTemplate: '<li><a href="{url}" title="Identification of unknown bacterial isolates using Sanger sequencing of the 16S rRNA gene">{title}</a></li>',
                    noResultsText: 'No results found.',
                            limit: 10,
                            fuzzy: true,
                    })
                    </script>
                    <!--end search-->
                </li>
            </ul>
        </div>
        </div>
        <!-- /.container -->
</nav>

<!-- Page Content -->
<div class="container">
    <div class="col-lg-12">&nbsp;</div>
    <!-- Content Row -->
    <div class="row">
        
        
            <!-- Sidebar Column -->
            <div class="col-md-3" id="tg-sb-sidebar">
                












<ul id="mysidebar" class="nav">
    <li class="sidebarTitle">CHMI services </li>
    
    
    
        
    
    <li>
        <a href="#">Overview</a>
        <ul>
            
            
            
            <li><a href="index.html">Welcome</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Bulk RNAseq</a>
        <ul>
            
            
            
            <li><a href="mydoc_RNAseq_FAQs.html">FAQs</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_TruSeq.html">mRNA library prep</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_total_TruSeq.html">Total transcriptome library prep</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_lowInput_RNAseq.html">Low input protocol</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_FFPE_RNAseq.html">FFPE-seq</a></li>
            
            
            
            <li class="subfolders">
                <a href="#">RNAseq data analysis</a>
                <ul>
                    
                    
                    
                    <li><a href="mydoc_RNAseq_QA.html">Project management and QA</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_kallisto.html">Read mapping with Kallisto</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_RNAseq_depositToGEO.html">Submitting data to GEO</a></li>
                    
                    
                    
                </ul>
            </li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Single cell genomics</a>
        <ul>
            
            
            
            <li><a href="mydoc_scRNAseq_FAQs.html">scRNAseq FAQs</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_10X_prep.html">10X sample prep</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_inDrop_encapsulation.html">inDrop encapsulation</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_inDrop_libraryPrep.html">inDrop library prep</a></li>
            
            
            
            <li class="subfolders">
                <a href="#">scRNAseq data analysis</a>
                <ul>
                    
                    
                    
                    <li><a href="mydoc_CellRanger.html">10X CellRanger</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_BUStools.html">BUStools</a></li>
                    
                    
                    
                </ul>
            </li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Microbiology</a>
        <ul>
            
            
            
            <li><a href="mydoc_micro_FAQs.html">Microbiology FAQs</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_16S.html">16S rRNA gene sequencing</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_NexteraXT.html">Shotgun metagenomics</a></li>
            
            
            
            
            
            
            <li class="active"><a href="mydoc_16S_Sanger.html">Sanger sequencing of 16S</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_mock.html">Culture collection</a></li>
            
            
            
            <li class="subfolders">
                <a href="#">Data analysis</a>
                <ul>
                    
                    
                    
                    <li><a href="mydoc_qiime2.html">QIIME2</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_microbiomeDB_deposit.html">microbiomeDB.org</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_sourmash.html">Sourmash</a></li>
                    
                    
                    
                </ul>
            </li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Compute resources</a>
        <ul>
            
            
            
            <li><a href="mydoc_linux.html">Linux cluster</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_software.html">Available software</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_RStudio.html">R Studio Server</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Cell culture</a>
        <ul>
            
            
            
            <li><a href="mydoc_3Dculture.html">3D intestinal cultures</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_3Dculture_microscopy.html">Immunofluorescence for 3D cultures</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">In the field</a>
        <ul>
            
            
            
            <li><a href="mydoc_nanopore.html">Oxford Nanopore</a></li>
            
            
            
            <li class="subfolders">
                <a href="#">Biomeme portable QPCR</a>
                <ul>
                    
                    
                    
                    <li><a href="mydoc_biomeme.html">Set-up</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_biomeme_avian.html">Avian</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_biomeme_human.html">Cutaneous leishmaniasis</a></li>
                    
                    
                    
                </ul>
            </li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Workshops</a>
        <ul>
            
            
            
            <li><a href="mydoc_cloudMeta.html">Metagenomics in the cloud</a></li>
            
            
            
            
        </ul>
     </li>
       
        
        
        <!-- if you aren't using the accordion, uncomment this block:
           <p class="external">
               <a href="#" id="collapseAll">Collapse All</a> | <a href="#" id="expandAll">Expand All</a>
           </p>
           -->
</ul>

<!-- this highlights the active parent class in the navgoco sidebar. this is critical so that the parent expands when you're viewing a page. This must appear below the sidebar code above. Otherwise, if placed inside customscripts.js, the script runs before the sidebar code runs and the class never gets inserted.-->
<script>$("li.active").parents('li').toggleClass("active");</script>

            </div>
            
        

        <!-- Content Column -->
        <div class="col-md-9" id="tg-sb-content">
            <div class="post-header">
   <h1 class="post-title-main">Identification of unknown bacterial isolates using Sanger sequencing of the 16S rRNA gene</h1>
</div>



<div class="post-content">

   
    <div class="summary">Identification of unknown bacterial isolates using Sanger sequencing of the 16S rRNA gene</div>
   

    
    
<!-- this handles the automatic toc. use ## for subheads to auto-generate the on-page minitoc. if you use html tags, you must supply an ID for the heading element in order for it to appear in the minitoc. -->
<script>
$( document ).ready(function() {
  // Handler for .ready() called.

$('#toc').toc({ minimumHeaders: 0, listType: 'ul', showSpeed: 0, headers: 'h2,h3,h4' });

/* this offset helps account for the space taken up by the floating toolbar. */
$('#toc').on('click', 'a', function() {
  var target = $(this.getAttribute('href'))
    , scroll_target = target.offset().top

  $(window).scrollTop(scroll_target - 10);
  return false
})
  
});
</script>

<div id="toc"></div>

    


    

    <a target="_blank" href="https://github.com/CHMI-SOPs/CHMI-SOPs.github.io/tree/master/pages/mydoc/mydoc_16S_Sanger.md" class="btn btn-default githubEditButton" role="button"><i class="fa fa-github fa-lg"></i> Edit me</a>

    

  <h2 id="before-starting">Before starting</h2>
<ul>
  <li>
    <p>The protocol uses three primers (two forward and one reverse) to generate amplicons for Sanger sequencing.  Combining the seqeunce data for the three amplicons generates a contiguous sequence that spans that majority of the full 16S rRNA gene, therby giving you a decent idea of bacterial identity. PCR reactions use the <a href="https://www.promega.com/-/media/files/resources/protocols/product-information-sheets/g/gotaq-green-master-mix-protocol.pdf">GoTaq Master Mix</a></p>
  </li>
  <li>
    <p>PCR Primers are as follows:</p>
  </li>
</ul>

<table>
  <thead>
    <tr>
      <th>name</th>
      <th>Sequence (5’ -&gt; 3’)</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>27fwd</td>
      <td>AGA GTT TGA TCM TGG CTC AG</td>
    </tr>
    <tr>
      <td>515fwd</td>
      <td>GTG CCA GCM GCC GCG GTA A</td>
    </tr>
    <tr>
      <td>1492rev</td>
      <td>CGG TTA CCT TGT TAC GAC TT</td>
    </tr>
  </tbody>
</table>

<h2 id="materials">Materials</h2>
<ul>
  <li>Promega GoTaq Green Master Mix (-20C)</li>
  <li>Molecular biology grade, nuclease-free water (-20C or room temperature)</li>
  <li>27F Primer</li>
  <li>1492R primer</li>
  <li>515F primer</li>
  <li>Ampure XP Beads (remove from fridge and acclimate at room temp for 30 min before use)</li>
  <li>100% Ethanol</li>
</ul>

<h2 id="step-1-obtain-genomic-dna">Step 1: Obtain genomic DNA</h2>

<ul>
  <li>Extract DNA using QIAGEN DNeasy kit (bacterial protocol) or other method of bacterial DNA extraction.</li>
  <li>Quantify DNA using the Qubit.</li>
</ul>

<h2 id="step-2-pcr-amplify-the-full-length-16s-rrna-gene">Step 2: PCR Amplify the full-length 16S rRNA gene</h2>

<ul>
  <li>Prepare the following, per sample, as a master mix:</li>
</ul>

<table>
  <thead>
    <tr>
      <th>Component</th>
      <th>Volume</th>
      <th>Final Concentration</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>GoTaq Green Master Mix, 2X</td>
      <td>25 uL</td>
      <td>1X</td>
    </tr>
    <tr>
      <td>27F primer, 100 uM</td>
      <td>0.2 uL</td>
      <td>0.2 uM</td>
    </tr>
    <tr>
      <td>1492R primer, 100 uM</td>
      <td>0.2 uL</td>
      <td>0.2 uM</td>
    </tr>
    <tr>
      <td>Nuclease-free water</td>
      <td>19.6 uL</td>
      <td>NA</td>
    </tr>
    <tr>
      <td>Total Volume</td>
      <td>45 uL</td>
      <td> </td>
    </tr>
  </tbody>
</table>

<div class="alert alert-info" role="alert"><i class="fa fa-info-circle"></i> <b>Note:</b> Make a master mix including all of the components except for the DNA.</div>

<div class="alert alert-info" role="alert"><i class="fa fa-info-circle"></i> <b>Note:</b> The original IDT stock solutions of the primers are at a concentration of 100 uM. Our primers are at this concentration in the freezer. If your primers are not this concentration, adjust the volume added.</div>

<ul>
  <li>
    <p>In a 96-well plate, pipette 45uL of prepared master mix per sample.</p>
  </li>
  <li>
    <p>Add 5uL of extracted DNA template (~150 ng template (&lt; 250 ng)) and pipette to mix. Each sample will have a total volume of 50uL.</p>
  </li>
  <li>
    <p>Place in thermocycler and run the following program under SANGER folder -&gt; 16SFULL (bold indicates 30 cycles)</p>
  </li>
</ul>

<table>
  <thead>
    <tr>
      <th>Temp (C)</th>
      <th>Time (min:sec)</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>95</td>
      <td>2:00</td>
    </tr>
    <tr>
      <td><strong>95</strong></td>
      <td><strong>0:30</strong></td>
    </tr>
    <tr>
      <td><strong>55</strong></td>
      <td><strong>0:30</strong></td>
    </tr>
    <tr>
      <td><strong>72</strong></td>
      <td><strong>1:40</strong></td>
    </tr>
    <tr>
      <td>72</td>
      <td>5:00</td>
    </tr>
    <tr>
      <td>12</td>
      <td>hold</td>
    </tr>
  </tbody>
</table>

<h2 id="step-3--cleanup">Step 3:  Cleanup</h2>

<ul>
  <li>
    <p>Vortex AMPure XP beads before each use.  Vortex AMPure XP beads frequently to make sure that beads are evenly distributed.</p>
  </li>
  <li>
    <p>Add 50 μl of AMPure XP beads to each well.</p>
  </li>
  <li>
    <p>Pipette to mix around 15 times to ensure the beads are mixed well with PCR products.</p>
  </li>
  <li>
    <p>Incubate at room temperature for 5 minutes.</p>
  </li>
  <li>
    <p>Place on a magnetic stand and wait until the liquid is clear (~2 minutes).</p>
  </li>
  <li>
    <p>Remove and discard all supernatant from each well.</p>
  </li>
  <li>Keeping the samples on the magnetic stand, wash 2 times as follows:
    <ul>
      <li>Add 180 μl fresh 80% EtOH to each well.</li>
      <li>Incubate on the magnetic stand for 30 seconds.</li>
      <li>Remove and discard all supernatant from each well.</li>
    </ul>
  </li>
  <li>
    <p>Using a 20 μl pipette, remove residual 80% EtOH from each well.</p>
  </li>
  <li>
    <p>Let the plate stand on the magnet at RT until dry, usually less than 5 minutes. When dry, the beads will appear matte and cracked.</p>
  </li>
  <li>
    <p>Remove from the magnetic stand when samples are dry.</p>
  </li>
  <li>
    <p>Add 52.5 μl RSB to each well.</p>
  </li>
  <li>
    <p>Pipette to mix well and resuspend beads.</p>
  </li>
  <li>
    <p>Incubate at room temperature for 2 minutes.</p>
  </li>
  <li>
    <p>Place on a magnetic stand and wait until the liquid is clear (~2 minutes).</p>
  </li>
  <li>Transfer 50 μl supernatant to a new plate.</li>
</ul>

<div class="alert alert-info" role="alert"><i class="fa fa-info-circle"></i> <b>Note:</b> This is a safe stopping point.  If you choose to stop here, seal the plate and store at -20°C for up to 7 days. Alternatively, leave in the thermal cycler overnight</div>

<h2 id="step-4-qc-on-purified-pcr-products">Step 4: QC on purified PCR products</h2>

<ul>
  <li>
    <p>Perform Qubit quantification on all samples</p>
  </li>
  <li>
    <p>Optional: Select a few samples and run an agarose gel or tapestation (DNA 5000 tape) to ensure there is a major product at approximately 1400 bp.</p>
  </li>
</ul>

<h2 id="step-5-submit-pcr-products-for-sequencing">Step 5: Submit PCR products for sequencing</h2>

<ul>
  <li>
    <p>Follow the <a href="https://www.med.upenn.edu/genetics/dnaseq/submission.shtml">submission guide instructions</a> carefully for submitting to the UPenn DNA sequencing facility</p>
  </li>
  <li>
    <p>Each tube should contain the following:</p>
    <ul>
      <li>6 uL of PCR product at 25 ng/uL</li>
      <li>3 uL of the desired primer at 1.1 uM</li>
      <li>If your DNA is not sufficiently concentrated, just submit the sample anyway, your results may be fine.</li>
      <li>The sequencing facility suggests 10 ng of DNA per 100 bp.</li>
    </ul>
  </li>
</ul>

<div class="alert alert-info" role="alert"><i class="fa fa-info-circle"></i> <b>Note:</b> We typically submit 3 tubes per sample: a 27F, 515F, and 1492R. These three sequencing results help me assemble the 16S gene without gaps.</div>

<div class="alert alert-info" role="alert"><i class="fa fa-info-circle"></i> <b>Note:</b> Use strip tubes with the strip caps for easy removal if you have &lt;52 samples. If you have more than 52 samples, submit your samples in a plate with the HT option.</div>

<div class="alert alert-warning" role="alert"><i class="fa fa-warning"></i> <b>Important:</b> The first tube of each 8-tube set should have ‘Beiting’ on it and the second tube should have your last name. Label your samples sequentially 1, 2, 3, etc….</div>

<h2 id="step-6-interpret-results">Step 6: Interpret results</h2>

<ul>
  <li>
    <p>Assemble the sequencing reads using your favorite assembly software.  We use Geneious (commercial software) because it has an easy-to-use interface.  .Ab1 files for all three amplicons can be dragged onto the Geneious interface and the <em>de novo assembly</em> tool is used to produce a consensus contig.</p>
  </li>
  <li>
    <p>Take the highest quality portion of the full length consensus sequence and seach for the best match using either <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE_TYPE=BlastSearch&amp;BLAST_SPEC=MicrobialGenomes">BLAST</a> or a recent tool called Bitsliced Genomic Signature Index (BIGSI) which is available <a href="http://www.bigsi.io/">here</a> and you can read more about on the <a href="https://github.com/Phelimb/BIGSI">Github page for BIGSI</a></p>
  </li>
</ul>

<div class="alert alert-warning" role="alert"><i class="fa fa-warning"></i> <b>Important:</b> No database is comprehensive and there’s always bias in what is represented in a database. In addition, since you’re only considering one gene (16S rRNA), this also affects your specificity and resolution for identifying species.</div>



    <div class="tags">
        
        <b>Tags: </b>
        
        
        
        <a href="tag_microbiome.html" class="btn btn-default navbar-btn cursorNorm" role="button">microbiome</a>
        
        
        
    </div>



    <div id="disqus_thread"></div>
    <script type="text/javascript">
        /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
        var disqus_shortname = 'http-hostmicrobe-org'; // required: replace example with your forum shortname

        /* * * DON'T EDIT BELOW THIS LINE * * */
        (function() {
            var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
            dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
            (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
        })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>


</div>

<hr class="shaded"/>

<footer>
            <div class="row">
                <div class="col-lg-12 footer">
               &copy;2020 PennVet Center for Host-Microbial Interactions. All rights reserved. <br />
 Site last generated: Mar 19, 2020 <br />
<p><img src="images/CHMI_logo.png" alt="Company logo"/></p>
                </div>
            </div>
</footer>


        </div>
    <!-- /.row -->
</div>
<!-- /.container -->
    </div>

</body>

<!-- the google_analytics_id gets auto inserted from the config file -->



<script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','//www.google-analytics.com/analytics.js','ga');ga('create','UA-131349832-1','auto');ga('require','displayfeatures');ga('send','pageview');</script>


</html>
