<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="description" content="Most of the projects we work on, whether profiling host gene expression or micobial communities, often yield at a candidate list of genes or organisms that w...">
<meta name="keywords" content="QPCRmobilediagnostics,  ">
<title>Portable QPCR | CHMI services</title>
<link rel="stylesheet" href="css/syntax.css">


<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
<!--<link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">-->
<link rel="stylesheet" href="css/modern-business.css">
<link rel="stylesheet" href="css/lavish-bootstrap.css">
<link rel="stylesheet" href="css/customstyles.css">
<link rel="stylesheet" href="css/theme-blue.css">

<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-cookie/1.4.1/jquery.cookie.min.js"></script>
<script src="js/jquery.navgoco.min.js"></script>


<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/2.0.0/anchor.min.js"></script>
<script src="js/toc.js"></script>
<script src="js/customscripts.js"></script>

<link rel="shortcut icon" href="images/favicon.ico">

<!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
<script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
<![endif]-->

<link rel="alternate" type="application/rss+xml" title="" href="http://localhost:4000/feed.xml">

    <script>
        $(document).ready(function() {
            // Initialize navgoco with default options
            $("#mysidebar").navgoco({
                caretHtml: '',
                accordion: true,
                openClass: 'active', // open
                save: false, // leave false or nav highlighting doesn't work right
                cookie: {
                    name: 'navgoco',
                    expires: false,
                    path: '/'
                },
                slide: {
                    duration: 400,
                    easing: 'swing'
                }
            });

            $("#collapseAll").click(function(e) {
                e.preventDefault();
                $("#mysidebar").navgoco('toggle', false);
            });

            $("#expandAll").click(function(e) {
                e.preventDefault();
                $("#mysidebar").navgoco('toggle', true);
            });

        });

    </script>
    <script>
        $(function () {
            $('[data-toggle="tooltip"]').tooltip()
        })
    </script>
    <script>
        $(document).ready(function() {
            $("#tg-sb-link").click(function() {
                $("#tg-sb-sidebar").toggle();
                $("#tg-sb-content").toggleClass('col-md-9');
                $("#tg-sb-content").toggleClass('col-md-12');
                $("#tg-sb-icon").toggleClass('fa-toggle-on');
                $("#tg-sb-icon").toggleClass('fa-toggle-off');
            });
        });
    </script>
    

</head>
<body>
<!-- Navigation -->
<nav class="navbar navbar-inverse navbar-fixed-top">
    <div class="container topnavlinks">
        <div class="navbar-header">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a class="fa fa-home fa-lg navbar-brand" href="index.html">&nbsp;<span class="projectTitle"> Center for Host-Microbial Interactions</span></a>
        </div>
        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
            <ul class="nav navbar-nav navbar-right">
                <!-- toggle sidebar button -->
                <li><a id="tg-sb-link" href="#"><i id="tg-sb-icon" class="fa fa-toggle-on"></i> Nav</a></li>
                <!-- entries without drop-downs appear here -->
                
                
                
                <!-- entries with drop-downs appear here -->
                <!-- conditional logic to control which topnav appears for the audience defined in the configuration file.-->
                
                
                
                
			<li>



  <a class="email" title="Submit feedback" href="#" onclick="javascript:window.location='mailto:beiting@upenn.edu?subject=CHMI services feedback&body=I have some feedback about the Portable QPCR page: ' + window.location.href;"><i class="fa fa-envelope-o"></i> Feedback</a>

</li>

		
                <!--comment out this block if you want to hide search-->
                <li>
                    <!--start search-->
                    <div id="search-demo-container">
                        <input type="text" id="search-input" placeholder="search...">
                        <ul id="results-container"></ul>
                    </div>
                    <script src="js/jekyll-search.js" type="text/javascript"></script>
                    <script type="text/javascript">
                            SimpleJekyllSearch.init({
                                searchInput: document.getElementById('search-input'),
                                resultsContainer: document.getElementById('results-container'),
                                dataSource: 'search.json',
                                searchResultTemplate: '<li><a href="{url}" title="Portable QPCR">{title}</a></li>',
                    noResultsText: 'No results found.',
                            limit: 10,
                            fuzzy: true,
                    })
                    </script>
                    <!--end search-->
                </li>
            </ul>
        </div>
        </div>
        <!-- /.container -->
</nav>

<!-- Page Content -->
<div class="container">
    <div class="col-lg-12">&nbsp;</div>
    <!-- Content Row -->
    <div class="row">
        
        
            <!-- Sidebar Column -->
            <div class="col-md-3" id="tg-sb-sidebar">
                












<ul id="mysidebar" class="nav">
    <li class="sidebarTitle">CHMI services </li>
    
    
    
        
    
    <li>
        <a href="#">Overview</a>
        <ul>
            
            
            
            <li><a href="index.html">Welcome</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">RNAseq</a>
        <ul>
            
            
            
            <li><a href="mydoc_RNAseq_FAQs.html">RNAseq FAQs</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_TruSeq.html">mRNA library prep</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_total_TruSeq.html">Total transcriptome library prep</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_lowInput_RNAseq.html">Low input protocol</a></li>
            
            
            
            <li class="subfolders">
                <a href="#">Data analysis</a>
                <ul>
                    
                    
                    
                    <li><a href="mydoc_linux.html">Compute resources</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_kallisto.html">Kallisto and Sleuth</a></li>
                    
                    
                    
                    
                    
                    <li><a href="tag_formatting.html">Data deposition</a></li>
                    
                    
                    
                </ul>
            </li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Microbiology</a>
        <ul>
            
            
            
            <li><a href="mydoc_micro_FAQs.html">Microbiology FAQs</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_16S.html">16S rRNA gene sequencing</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_NexteraXT.html">Shotgun metagenomics</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_16S_Sanger.html">Sanger sequencing of 16S</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_mock.html">Culture collection</a></li>
            
            
            
            <li class="subfolders">
                <a href="#">Data analysis</a>
                <ul>
                    
                    
                    
                    <li><a href="mydoc_linux.html">Compute resources</a></li>
                    
                    
                    
                    
                    
                    <li><a href="tag_formatting.html">MicrobiomeDB</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_qiime2.html">QIIME2</a></li>
                    
                    
                    
                    
                    
                    <li><a href="tag_formatting.html">Data deposition</a></li>
                    
                    
                    
                </ul>
            </li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Cell culture</a>
        <ul>
            
            
            
            <li><a href="mydoc_3Dculture.html">3D intestinal cultures</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">In the field</a>
        <ul>
            
            
            
            <li class="active"><a href="mydoc_biomeme.html">Biomeme portable QPCR</a></li>
            
            
            
            
        </ul>
     </li>
       
        
        
        <!-- if you aren't using the accordion, uncomment this block:
           <p class="external">
               <a href="#" id="collapseAll">Collapse All</a> | <a href="#" id="expandAll">Expand All</a>
           </p>
           -->
</ul>

<!-- this highlights the active parent class in the navgoco sidebar. this is critical so that the parent expands when you're viewing a page. This must appear below the sidebar code above. Otherwise, if placed inside customscripts.js, the script runs before the sidebar code runs and the class never gets inserted.-->
<script>$("li.active").parents('li').toggleClass("active");</script>

            </div>
            
        

        <!-- Content Column -->
        <div class="col-md-9" id="tg-sb-content">
            <div class="post-header">
   <h1 class="post-title-main">Portable QPCR</h1>
</div>



<div class="post-content">

   
    <div class="summary">Most of the projects we work on, whether profiling host gene expression or micobial communities, often yield at a candidate list of genes or organisms that we believe are linked to some phenotype.  Increasingly, we are interested in methods and technology that let us quantitiatively measure these features, particularly in a way that is portable and could be deployed 'stall-side' in a barn just as easily as at a lab bench.  For this, we've been using the Biomeme platform for end-to-end QPCR-based detection in the field</div>
   

    
    
<!-- this handles the automatic toc. use ## for subheads to auto-generate the on-page minitoc. if you use html tags, you must supply an ID for the heading element in order for it to appear in the minitoc. -->
<script>
$( document ).ready(function() {
  // Handler for .ready() called.

$('#toc').toc({ minimumHeaders: 0, listType: 'ul', showSpeed: 0, headers: 'h2,h3,h4' });

/* this offset helps account for the space taken up by the floating toolbar. */
$('#toc').on('click', 'a', function() {
  var target = $(this.getAttribute('href'))
    , scroll_target = target.offset().top

  $(window).scrollTop(scroll_target - 10);
  return false
})
  
});
</script>

<div id="toc"></div>

    


    

    <a target="_blank" href="https://github.com/CHMI-SOPs/CHMI-SOPs.github.io/tree/master/pages/mydoc/mydoc_biomeme.md" class="btn btn-default githubEditButton" role="button"><i class="fa fa-github fa-lg"></i> Edit me</a>

    

  <h2 id="before-starting">Before starting</h2>
<p>The protocols below detail how we’ve been using <a href="http://biomeme.com/">Biomeme Inc’s</a> bulk reagents for QPCRs.  Once you’ve optimized the assays using bulk reagents, assays can then be manufactured as room-temp stable QPCR assays for use in the field.  For the most part, we keep our mastermix and PCR conditions the same across all assays.  The variations come with the primer/probe design and optimizing extaction of DNA or RNA from different sample types.  So far, we’ve had good success with the primers/probes listed below and with extraction of nucleic acid from nasal and skin swabs.</p>

<div class="alert alert-info" role="alert"><i class="fa fa-info-circle"></i> <b>Note:</b> The only difference between DNA and RNA protocols are the BLB buffer in the extraction kit, and the type of master mix you will use (LyoDNA versus LyoRNA).</div>

<h2 id="devices">Devices</h2>
<p>The hand-held QPCR device made by Biomeme, called the ‘Two3’, can accomodate three PCR tubes and can detect two colors in each tube: FAM and Cy5 (or ATTO647N, which is brighter than Cy5, but more costly to order).  Biomeme will soon be releasing the ‘three9’ which, as the name suggests, accomodates 9 tubes and three colors/tube: FAM, Cy5/ATTO647N and TexasRed.  ATTO647N is the brightest channel on the three9, so consider good designing the target you anticipate being present in the lowest concentration and/or least sensitive PCR to that channel. FAM is the next brightest, followed by TexasRedX. If you are designing a triplex assay for use with the three9 that includes a positive control, use TexasRedX for the positive control.</p>

<h2 id="preparing-mastermix">Preparing mastermix</h2>
<p>Resuspend bulk lyophilized <a href="http://CHMI-sops.github.io/papers/LyoDNA.pdf">LyoDNA</a>, <a href="http://CHMI-sops.github.io/papers/LyoRNA.pdf">LyoRNA</a> or <a href="http://CHMI-sops.github.io/papers/LyoGreen.pdf">LyoGreen</a> master mix with 135 ul of PCR grade water + 18 ul of glycerol.  This is a 10x stock.  Addition of glyceral allows leftover mastermix to be frozen for reuse later.  LyoRNA master mix contains a reverse transcriptase for QPCR assays where the target is a RNA molecule.  Each resuspended vial contains enough mastermix for 60 reactions of 20ul/rxn.</p>

<h2 id="reaction-set-up-with-lyodna-or-lyorna">Reaction set-up with LyoDNA or LyoRNA</h2>
<p>A typical QPCR reaction on the biomeme platform contains the following components</p>

<table>
  <thead>
    <tr>
      <th>Reagent</th>
      <th>vol (ul)</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>20x primer/probe</td>
      <td>1</td>
    </tr>
    <tr>
      <td>10x LyoDNA, LyoRNA or LyoGreen mastermix</td>
      <td>2</td>
    </tr>
    <tr>
      <td>isolated DNA or RNA</td>
      <td>5-17</td>
    </tr>
    <tr>
      <td>water</td>
      <td>qs to 20</td>
    </tr>
  </tbody>
</table>

<div class="alert alert-warning" role="alert"><i class="fa fa-warning"></i> <b>Important:</b> 20x primer/probe stock contains 4uM of the forward primer and probe, and 8uM of the reverse primer in TE pH 8.0.  Final concentrations in the PCR reaction will be 200nM of forward and probe, 400nM of reverse.  If starting with 100 uM stocks, then add 84 uL TE, 4 uL of forward, 4 uL of probe and 8 uL of reverse, to get a 100 ul stock of 20x</div>

<h2 id="run-set-up">Run set-up</h2>

<ul>
  <li>To begin, launch the Biomeme app on the iPhone connected to your device.</li>
  <li>To run any assays, you’ll need to set up a data folder by selecting the ‘data management’ button on the main screen.  Name your folder, create any subfolders if necessary and save.</li>
  <li>Back on the main screen, choose ‘protocol management’ to set up a protocol that will contain your thermocycling conditions and a standard curve (optional).  We’ve always used the standard thermocycling conditions that come pre-set on the device as follows:</li>
</ul>

<table>
  <thead>
    <tr>
      <th>Temp (C)</th>
      <th>Time (min:sec)</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>45</td>
      <td>10:00</td>
    </tr>
    <tr>
      <td>95</td>
      <td>10:00</td>
    </tr>
    <tr>
      <td><strong>95</strong></td>
      <td><strong>0:15</strong></td>
    </tr>
    <tr>
      <td><strong>60</strong></td>
      <td><strong>1:00</strong></td>
    </tr>
  </tbody>
</table>

<p><strong>Bold</strong> steps are repeated for 40 cycles total</p>

<ul>
  <li>Give your protocol a name and select ‘save’ in the upper right corner of the app</li>
  <li>If you want to set-up a standard curve, select your protocol, scroll to the bottom, and select standard curve.</li>
</ul>

<div class="alert alert-info" role="alert"><i class="fa fa-info-circle"></i> <b>Note:</b> standard curves are specific to each device.  If you plan to run the same assay across multiple devices, you’ll have to run the standard curve on each device.</div>

<ul>
  <li>
    <p>Currently, standards can only have three points.  Enter the values for your 3 point standard curve.  Since the Two3 device sees a red and green channel, you have to enter 3 standard values for each channel.  Enter zero or one for each point in the channel you’re not using.</p>
  </li>
  <li>
    <p>To begin a run, go back to the main screen of the app and choose ‘start run’.  Choose your data folder, then your protocol, then select the run type (standard curve, qualitative, etc). Hit run and follow the on screen instructions to begin the run</p>
  </li>
</ul>

<div class="alert alert-danger" role="alert"><i class="fa fa-exclamation-circle"></i> <b>Warning:</b> once you begin a run on the Biomeme Two3, be sure not to hit the home button on the iphone.  Doing so will immediately close the app and result in terminating the run.  We place tape over the home button with a ‘do not touch’ label.</div>

<h2 id="avian-assays">Avian assays</h2>

<h3 id="tracheal-swabs">Tracheal swabs</h3>
<p>All avian assays begin with tracheal swabs collected in brain-heart infusion (BHI) media.  We typically have 11 swabs all stored in a single vial containing 5.5 ml of BHI media.  These swabs could be from multiple birds or from a single bird sampled multiple times, and this vial can be assayed fresh or after storage at -80C.  Here’s our procotol for rapid, 1 minute DNA extraction using Biomeme reagents:</p>

<ol>
  <li>thaw vial containing BHI media and swabs</li>
  <li>take 100 ul aliquot of sample for DNA or RNA extraction (might be quite viscous due to mucous)</li>
  <li>add 100 ul sample to 1 ml of Biomeme Lysis Buffer (BLB) (if you plan to assay for IBV, use BLB from RNA kit)</li>
  <li>with a 1cc syringe affixed to a Biomeme extraction column, push the extraction sample mix up and down 10 times</li>
  <li>after expelling the volume the 10th time, wash the column by drawing up 500 ul of Biomeme Protein Wash (BPW) 1 time, then expel again.</li>
  <li>draw up 1 ml of Biomeme Wash Buffer (BWB) and expel volume</li>
  <li>draw up 1 ml of Biomeme Drying Wash (BDW) and expel volume</li>
  <li>remove and discard the 1cc syringe, and attach a new 50cc syringe to the same column.  Pump up and down with air vigorously to dry the column.</li>
  <li>remove and discard the 50cc syringe.  Connect new 1 cc syringe to the column and draw up 125 ul of Biomeme Elution Buffer (BEB).</li>
  <li>Expel the elution buffer into a final collection tube.  Typical elution volume is 50-70 ul.  We use 5ul of this DNA for any of the assays listed below, which means a single eluate is enough for at least 9-10 QPCR assays.</li>
</ol>

<h3 id="primers-and-probes">Primers and probes</h3>

<h4 id="infectious-bronchitis-virus-ibv">Infectious Bronchitis Virus (IBV)</h4>

<div class="alert alert-info" role="alert"><i class="fa fa-info-circle"></i> <b>Note:</b> IBV is an RNA virus, so be sure to use the appropriate lysis buffer (BLB from RNA bulk extraction kit) and mastermix (LyoRNA).</div>

<table>
  <thead>
    <tr>
      <th>name</th>
      <th>Sequence (5’ -&gt; 3’)</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>fwd</td>
      <td>GCT TTT GAG CCT AGC GTT</td>
    </tr>
    <tr>
      <td>rev</td>
      <td>GCC ATG TTG TCA CTG TCT ATT G</td>
    </tr>
    <tr>
      <td>probe</td>
      <td><em>FAM</em>-CAC CAC CAG AAC CTG TCA CCT C-<em>BHQ-1</em></td>
    </tr>
  </tbody>
</table>

<h4 id="mycoplasma-gallisepticum-and-mycoplasma-synoviae-mgms"><em>Mycoplasma gallisepticum</em> and <em>Mycoplasma synoviae</em> (Mg/Ms)</h4>

<div class="alert alert-info" role="alert"><i class="fa fa-info-circle"></i> <b>Note:</b> Mg and Ms targets are DNA, so be sure to use appropirate lysis buffer (BLB from DNA bulk extraction kit) and mastermix (LyoDNA).</div>

<p>The primer and probe sequences below are from <a href="http://www.bioone.org/doi/10.1637/8469-091508-Reg.1?url_ver=Z39.88-2003&amp;rfr_id=ori:rid:crossref.org&amp;rfr_dat=cr_pub%3dpubmed">Raviv and Kleven, 2009</a></p>

<table>
  <thead>
    <tr>
      <th>name</th>
      <th>Sequence (5’ -&gt; 3’)</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Mg fwd</td>
      <td>TTG GGT TTA GGG ATT GGGA TT</td>
    </tr>
    <tr>
      <td>Mg rev</td>
      <td>CCA AGGG ATT CAA CCA TCT T</td>
    </tr>
    <tr>
      <td>Mg probe</td>
      <td><em>5TexRd-XN</em>-TGA TGA TCC AAG AAC GTG AAG AAC ACC <em>BHQ-2</em></td>
    </tr>
    <tr>
      <td>Ms fwd</td>
      <td>CTA AAT ACA ATA GCC CAA GGC AA</td>
    </tr>
    <tr>
      <td>Ms rev</td>
      <td>CCT CCT TTC TTA CGG AGT ACA</td>
    </tr>
    <tr>
      <td>Ms probe</td>
      <td><em>FAM</em>-AGC GAT ACA CAA CCG CTT TTA GAA T-<em>BHQ-1</em></td>
    </tr>
  </tbody>
</table>

<h2 id="cutaneous-leishmaniasis-assay">Cutaneous Leishmaniasis assay</h2>

<h3 id="lesion-swab">Lesion swab</h3>
<p>For each patient, two sterile swabs (provided by Biomeme) are held together and used to swab the inner border of lesion (encircled 20x).  This will be used for DNA targets (bacteria).  Repeat with two new swabs (20x), and use this second set of swabs for RNA targets (host genes).</p>

<ol>
  <li>Immerse swabs intended for DNA in 1ml of a 1:1 mix of BLB from DNA kit (0.5ml) and water (BEB; 0.5ml), twirl in media for 1 min, discard swabs.</li>
  <li>Immerse swabs intended for RNA in 1ml of BLB from the RNA kit, twirl in media for 1min, discard swabs</li>
  <li>Attach standard lab 1cc syringe to Biomeme filter tip. Pump lysate up/down 10x, expel volume</li>
  <li>Immediately move to <strong>BPW</strong> wash (1ml), pump up/down 1x, expel volume</li>
  <li>Immediately move to <strong>BWB</strong> wash (1ml), pump up/down 1x, expel volume</li>
  <li>Immediately move to <strong>BDW</strong> wash (1ml), pump up/down 1x, expel volume</li>
  <li>Pump air up/down repeatedly 10x, discard syringe but <strong>keep filter tip</strong></li>
  <li>attach 25cc or 50cc syringe to the same filter tip.  Pump up and down with air vigorously to dry the column.</li>
  <li>remove and discard the syringe.  Connect new Biomeme ‘slip-tip’ 1 cc syringe to the column and draw up 150 ul of Biomeme Elution Buffer (BEB).  Pump up/down 3x with BEB.</li>
  <li>Expel the elution buffer into a final collection tube.  Typical elution volume is ~120ul.  We use 17ul of this DNA for any of the assays listed below, which means a single eluate is enough for at least 6 QPCR assays.</li>
</ol>

<h3 id="primers-and-probes-1">Primers and probes</h3>

<h4 id="skin-resident-microbes">Skin-resident microbes</h4>
<p>The primers and probes below are used for detection of <em>Staphylococcus aureus (Saur)</em>, <em>Streptococcus pyogenes (Spyo)</em> and <em>Leishmania braziliensis (Lbra)</em></p>

<div class="alert alert-info" role="alert"><i class="fa fa-info-circle"></i> <b>Note:</b> All of these targets are DNA, so be sure to use appropirate lysis buffer (BLB from DNA bulk extraction kit) and mastermix (LyoDNA).</div>

<div class="alert alert-info" role="alert"><i class="fa fa-info-circle"></i> <b>Note:</b> We prepare a standard curve for the <em>S. aureus</em> assay using genomic DNA provided by Biomeme.  Stock is ~100,000 <em>S. aureus</em> genomes per ul, and curve is 100000, 10000 and 1000 copies.  We prepare a similar standard curve from <em>L. braziliensis</em> promastigotes, with a stock that is also ~100,000 genome copies per ul.  When running a standard curve on the Two3 device, always put the tube containing the lowest amount of target in the center well.  This position is closest to the sensor, and therefore provides a bit more sensitivity.</div>

<ul>
  <li><em>Leishmania braziliensis</em> primer/probes are from <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3209110/">Weirather et al., 2011</a></li>
  <li><em>Streptococcus pyogenes</em> primer/probes were adapted from CDC <a href="https://www.cdc.gov/streplab/protocols.html">here</a></li>
  <li><em>Staphylococcus aureus</em> primers and probes were provided by Biomeme Inc.</li>
</ul>

<table>
  <thead>
    <tr>
      <th>name</th>
      <th>Sequence (5’ -&gt; 3’)</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><em>Saur</em> fwd</td>
      <td>provided by Biomeme</td>
    </tr>
    <tr>
      <td><em>Saur</em> rev</td>
      <td>provided by Biomeme</td>
    </tr>
    <tr>
      <td><em>Saur</em> probe</td>
      <td>provided by Biomeme</td>
    </tr>
    <tr>
      <td><em>Spyo</em> fwd</td>
      <td>GCA CTC GCT ACT ATT TCT TAC CTC AA</td>
    </tr>
    <tr>
      <td><em>Spyo</em> rev</td>
      <td>ATT ACT GGT TTC CAA GAC ATT GTG AC</td>
    </tr>
    <tr>
      <td><em>Spyo</em> probe</td>
      <td><em>/56-FAM/</em>CCG CAA CTC <em>/ZEN/</em> ATC AAG GAT TTC TGT TAC CA<em>/3IABkFQ/</em></td>
    </tr>
    <tr>
      <td><em>Lbra</em> fwd</td>
      <td>TGC TAT AAA ATC GTA CCA CCC GAC A</td>
    </tr>
    <tr>
      <td><em>Lbra</em> rev</td>
      <td>AAA TGG CAT ACA GAA ACC CCG TTC</td>
    </tr>
    <tr>
      <td><em>Lbra</em> probe</td>
      <td><em>/56-FAM/</em>GCC TCT GGG <em>/ZEN/</em> TAG GGG CGT TCT GCA A<em>/3IABkFQ/</em></td>
    </tr>
  </tbody>
</table>

<h4 id="human-transcripts">Human transcripts</h4>
<p>The primers and probes below are for the detection of host transcripts present in cellular material on recovered from the swab</p>

<div class="alert alert-info" role="alert"><i class="fa fa-info-circle"></i> <b>Note:</b> All of these targets are mRNAs, so be sure to use appropirate lysis buffer (BLB from RNA bulk extraction kit) and mastermix (LyoRNA).</div>

<div class="alert alert-info" role="alert"><i class="fa fa-info-circle"></i> <b>Note:</b> The 18S assay is included as a positive extraction control.  The primers recognize an exon in the 18S gene, so there will be some signal from DNA in the reaction (the Biomeme RNA extraction isolates both RNA and DNA), however, given the large copy number for 18S transcripts, there should be much stronger signal for RNA.</div>

<table>
  <thead>
    <tr>
      <th>name</th>
      <th>Sequence (5’ -&gt; 3’)</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>hIL1B fwd</td>
      <td>CAA AGG CGG CCA GGA TAT AA</td>
    </tr>
    <tr>
      <td>hIL1B rev</td>
      <td>CTA GGG ATT GAG TCC ACA TTC AG</td>
    </tr>
    <tr>
      <td>hIL1B probe</td>
      <td><em>/56-FAM/</em>AGA GCT GTA <em>/ZEN/</em> CCC AGA GAG TCC TGT<em>/3IABkFQ/</em></td>
    </tr>
    <tr>
      <td>hGZMB fwd</td>
      <td>GTA CCA TTG AGT TGT GCG TG</td>
    </tr>
    <tr>
      <td>hGZMB rev</td>
      <td>CAT GCC ATT GTT TCG TCC ATA G</td>
    </tr>
    <tr>
      <td>hGZMB probe</td>
      <td><em>/56-FAM/</em>CCT CCA GAG <em>/ZEN/</em> TCC CCC TTA AAG GAA GT<em>/3IABkFQ/</em></td>
    </tr>
    <tr>
      <td>h18S (+ control) fwd</td>
      <td>provided by Biomeme</td>
    </tr>
    <tr>
      <td>h18S (+ control) rev</td>
      <td>provided by Biomeme</td>
    </tr>
    <tr>
      <td>h18S (+ control) probe</td>
      <td>provided by Biomeme</td>
    </tr>
  </tbody>
</table>



    <div class="tags">
        
        <b>Tags: </b>
        
        
        
        
        
        <a href="tag_mobile.html" class="btn btn-default navbar-btn cursorNorm" role="button">mobile</a>
        
        
        
        
        
    </div>



    <div id="disqus_thread"></div>
    <script type="text/javascript">
        /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
        var disqus_shortname = 'http-hostmicrobe-org'; // required: replace example with your forum shortname

        /* * * DON'T EDIT BELOW THIS LINE * * */
        (function() {
            var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
            dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
            (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
        })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>


</div>

<hr class="shaded"/>

<footer>
            <div class="row">
                <div class="col-lg-12 footer">
               &copy;2018 PennVet Center for Host-Microbial Interactions. All rights reserved. <br />
 Site last generated: May 16, 2018 <br />
<p><img src="images/CHMI_logo.png" alt="Company logo"/></p>
                </div>
            </div>
</footer>


        </div>
    <!-- /.row -->
</div>
<!-- /.container -->
    </div>

</body>

</html>
