<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="description" content="Kallisto is our first choice for aligning reads because it combines speed, accuracy, and the ability to leverage bootstraps for modeling technical variance">
<meta name="keywords" content="bioinformaticstranscriptomics,  ">
<title>Kallisto | CHMI services</title>
<link rel="stylesheet" href="css/syntax.css">


<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
<!--<link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">-->
<link rel="stylesheet" href="css/modern-business.css">
<link rel="stylesheet" href="css/lavish-bootstrap.css">
<link rel="stylesheet" href="css/customstyles.css">
<link rel="stylesheet" href="css/theme-blue.css">

<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-cookie/1.4.1/jquery.cookie.min.js"></script>
<script src="js/jquery.navgoco.min.js"></script>


<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/2.0.0/anchor.min.js"></script>
<script src="js/toc.js"></script>
<script src="js/customscripts.js"></script>

<link rel="shortcut icon" href="images/favicon.ico">

<!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
<script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
<![endif]-->

<link rel="alternate" type="application/rss+xml" title="" href="http://localhost:9000/feed.xml">



    <script>
        $(document).ready(function() {
            // Initialize navgoco with default options
            $("#mysidebar").navgoco({
                caretHtml: '',
                accordion: true,
                openClass: 'active', // open
                save: false, // leave false or nav highlighting doesn't work right
                cookie: {
                    name: 'navgoco',
                    expires: false,
                    path: '/'
                },
                slide: {
                    duration: 400,
                    easing: 'swing'
                }
            });

            $("#collapseAll").click(function(e) {
                e.preventDefault();
                $("#mysidebar").navgoco('toggle', false);
            });

            $("#expandAll").click(function(e) {
                e.preventDefault();
                $("#mysidebar").navgoco('toggle', true);
            });

        });

    </script>
    <script>
        $(function () {
            $('[data-toggle="tooltip"]').tooltip()
        })
    </script>
    <script>
        $(document).ready(function() {
            $("#tg-sb-link").click(function() {
                $("#tg-sb-sidebar").toggle();
                $("#tg-sb-content").toggleClass('col-md-9');
                $("#tg-sb-content").toggleClass('col-md-12');
                $("#tg-sb-icon").toggleClass('fa-toggle-on');
                $("#tg-sb-icon").toggleClass('fa-toggle-off');
            });
        });
    </script>
    

</head>
<body>
<!-- Navigation -->
<nav class="navbar navbar-inverse navbar-fixed-top">
    <div class="container topnavlinks">
        <div class="navbar-header">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a class="fa fa-home fa-lg navbar-brand" href="index.html">&nbsp;<span class="projectTitle"> Center for Host-Microbial Interactions</span></a>
        </div>
        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
            <ul class="nav navbar-nav navbar-right">
                <!-- toggle sidebar button -->
                <li><a id="tg-sb-link" href="#"><i id="tg-sb-icon" class="fa fa-toggle-on"></i> Nav</a></li>
                <!-- entries without drop-downs appear here -->
                
                
                
                <!-- entries with drop-downs appear here -->
                <!-- conditional logic to control which topnav appears for the audience defined in the configuration file.-->
                
                
                
                
			<li>



  <a class="email" title="Submit feedback" href="#" onclick="javascript:window.location='mailto:beiting@upenn.edu?subject=CHMI services feedback&body=I have some feedback about the Kallisto page: ' + window.location.href;"><i class="fa fa-envelope-o"></i> Feedback</a>

</li>

		
                <!--comment out this block if you want to hide search-->
                <li>
                    <!--start search-->
                    <div id="search-demo-container">
                        <input type="text" id="search-input" placeholder="search...">
                        <ul id="results-container"></ul>
                    </div>
                    <script src="js/jekyll-search.js" type="text/javascript"></script>
                    <script type="text/javascript">
                            SimpleJekyllSearch.init({
                                searchInput: document.getElementById('search-input'),
                                resultsContainer: document.getElementById('results-container'),
                                dataSource: 'search.json',
                                searchResultTemplate: '<li><a href="{url}" title="Kallisto">{title}</a></li>',
                    noResultsText: 'No results found.',
                            limit: 10,
                            fuzzy: true,
                    })
                    </script>
                    <!--end search-->
                </li>
            </ul>
        </div>
        </div>
        <!-- /.container -->
</nav>

<!-- Page Content -->
<div class="container">
    <div class="col-lg-12">&nbsp;</div>
    <!-- Content Row -->
    <div class="row">
        
        
            <!-- Sidebar Column -->
            <div class="col-md-3" id="tg-sb-sidebar">
                












<ul id="mysidebar" class="nav">
    <li class="sidebarTitle">CHMI services </li>
    
    
    
        
    
    <li>
        <a href="#">Overview</a>
        <ul>
            
            
            
            <li><a href="index.html">Welcome</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Bulk RNAseq</a>
        <ul>
            
            
            
            <li><a href="mydoc_RNAseq_FAQs.html">FAQs</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_TruSeq.html">mRNA library prep</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_total_TruSeq.html">Total transcriptome library prep</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_lowInput_RNAseq.html">Low input protocol</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_FFPE_RNAseq.html">FFPE-seq</a></li>
            
            
            
            <li class="subfolders">
                <a href="#">RNAseq data analysis</a>
                <ul>
                    
                    
                    
                    <li><a href="mydoc_RNAseq_QA.html">Project management and QA</a></li>
                    
                    
                    
                    
                    
                    <li class="active"><a href="mydoc_kallisto.html">Read mapping with Kallisto</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_RNAseq_depositToGEO.html">Submitting data to GEO</a></li>
                    
                    
                    
                </ul>
            </li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Single cell genomics</a>
        <ul>
            
            
            
            <li><a href="mydoc_scRNAseq_FAQs.html">scRNAseq FAQs</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_10X_prep.html">10X sample prep</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_inDrop_encapsulation.html">inDrop encapsulation</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_inDrop_libraryPrep.html">inDrop library prep</a></li>
            
            
            
            <li class="subfolders">
                <a href="#">scRNAseq data analysis</a>
                <ul>
                    
                    
                    
                    <li><a href="mydoc_CellRanger.html">10X CellRanger</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_CellRanger.html">BUStools</a></li>
                    
                    
                    
                </ul>
            </li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Microbiology</a>
        <ul>
            
            
            
            <li><a href="mydoc_micro_FAQs.html">Microbiology FAQs</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_16S.html">16S rRNA gene sequencing</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_NexteraXT.html">Shotgun metagenomics</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_16S_Sanger.html">Sanger sequencing of 16S</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_mock.html">Culture collection</a></li>
            
            
            
            <li class="subfolders">
                <a href="#">Data analysis</a>
                <ul>
                    
                    
                    
                    <li><a href="mydoc_qiime2.html">QIIME2</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_microbiomeDB_deposit.html">microbiomeDB.org</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_sourmash.html">Sourmash</a></li>
                    
                    
                    
                </ul>
            </li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Compute resources</a>
        <ul>
            
            
            
            <li><a href="mydoc_linux.html">Linux cluster</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_software.html">Available software</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_RStudio.html">R Studio Server</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Cell culture</a>
        <ul>
            
            
            
            <li><a href="mydoc_3Dculture.html">3D intestinal cultures</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_3Dculture_microscopy.html">Immunofluorescence for 3D cultures</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Portable sequencing</a>
        <ul>
            
            
            
            <li><a href="mydoc_biomeme.html">ONT MinION set-up</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Portable QPCR</a>
        <ul>
            
            
            
            <li><a href="mydoc_biomeme.html">Biomeme setup</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_biomeme_avian.html">Avian</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_biomeme_human.html">Cutaneous leishmaniasis</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Workshops</a>
        <ul>
            
            
            
            <li><a href="mydoc_cloudMeta.html">Metagenomics in the cloud</a></li>
            
            
            
            
        </ul>
     </li>
       
        
        
        <!-- if you aren't using the accordion, uncomment this block:
           <p class="external">
               <a href="#" id="collapseAll">Collapse All</a> | <a href="#" id="expandAll">Expand All</a>
           </p>
           -->
</ul>

<!-- this highlights the active parent class in the navgoco sidebar. this is critical so that the parent expands when you're viewing a page. This must appear below the sidebar code above. Otherwise, if placed inside customscripts.js, the script runs before the sidebar code runs and the class never gets inserted.-->
<script>$("li.active").parents('li').toggleClass("active");</script>

            </div>
            
        

        <!-- Content Column -->
        <div class="col-md-9" id="tg-sb-content">
            <div class="post-header">
   <h1 class="post-title-main">Kallisto</h1>
</div>



<div class="post-content">

   
    <div class="summary">Kallisto is our first choice for aligning reads because it combines speed, accuracy, and the ability to leverage bootstraps for modeling technical variance</div>
   

    
    
<!-- this handles the automatic toc. use ## for subheads to auto-generate the on-page minitoc. if you use html tags, you must supply an ID for the heading element in order for it to appear in the minitoc. -->
<script>
$( document ).ready(function() {
  // Handler for .ready() called.

$('#toc').toc({ minimumHeaders: 0, listType: 'ul', showSpeed: 0, headers: 'h2,h3,h4' });

/* this offset helps account for the space taken up by the floating toolbar. */
$('#toc').on('click', 'a', function() {
  var target = $(this.getAttribute('href'))
    , scroll_target = target.offset().top

  $(window).scrollTop(scroll_target - 10);
  return false
})
  
});
</script>

<div id="toc"></div>

    


    

    <a target="_blank" href="https://github.com/CHMI-SOPs/CHMI-SOPs.github.io/tree/master/pages/mydoc/mydoc_kallisto.md" class="btn btn-default githubEditButton" role="button"><i class="fa fa-github fa-lg"></i> Edit me</a>

    

  <h2 id="intro">Intro</h2>
<p><a href="https://pachterlab.github.io/kallisto/about">Kallisto</a> is a relatively new tool from Lior Pachter’s lab at UC Berkeley and is described in <a href="http://CHMI-sops.github.io/papers/Kallisto.pdf">this 2016 Nature Biotechnology paper</a>. Kallisto and other tools like it (e.g. <a href="https://combine-lab.github.io/salmon/">Salmon</a>) have revolutionized the analysis of RNAseq data by using extremely lightweight ‘pseudomapping’ that effectively allows analyses to be carried out on a standard laptop.  If you are reluctant to try pseudomapping out of concern that it won’t produce accurate quantifications of transcripts, <a href="https://www.nature.com/articles/s41467-017-00050-4">rest assured it will</a>.</p>

<h2 id="installing-kallisto-on-a-mac-os">Installing Kallisto on a Mac OS</h2>

<p>If you’re running a Mac OS, then begin by downloading and installing <a href="https://brew.sh/">Homebrew</a> with this single line of code:</p>

<div class="language-shell highlighter-rouge"><pre class="highlight"><code>/usr/bin/ruby -e <span class="s2">"</span><span class="k">$(</span>curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install<span class="k">)</span><span class="s2">"</span>
</code></pre>
</div>

<p>Now, installing Kallisto is simple:</p>

<div class="language-shell highlighter-rouge"><pre class="highlight"><code>brew tap homebrew/core
brew install kallisto
</code></pre>
</div>

<p>Test whether Kallisto is properly installed by typing <code class="highlighter-rouge">kallisto</code>, and you should see this output</p>

<div class="language-shell highlighter-rouge"><pre class="highlight"><code>kallisto 0.45.0

Usage: kallisto &lt;CMD&gt; <span class="o">[</span>arguments] ..

Where &lt;CMD&gt; can be one of:

    index         Builds a kallisto index 
    quant         Runs the quantification algorithm 
    bus           Generate BUS files <span class="k">for </span>single cell data 
    pseudo        Runs the pseudoalignment step 
    merge         Merges several batch runs 
    h5dump        Converts HDF5-formatted results to plaintext
    inspect       Inspects and gives information about an index
    version       Prints version information
    cite          Prints citation information

Running kallisto &lt;CMD&gt; without arguments prints usage information <span class="k">for</span> &lt;CMD&gt;
</code></pre>
</div>

<h2 id="installing-kallisto-on-a-windows-os">Installing Kallisto on a Windows OS</h2>

<ol>
  <li>Obtain administrative access for your computer</li>
  <li>You will need to be able to unzip files.  If you don’t have software that can do this, you can download <a href="https://www.win-rar.com/start.html?&amp;L=0">WinRAR</a> and install it in Program Files.</li>
  <li>Download the latest windows release of <a href="https://pachterlab.github.io/kallisto/download">Kallisto</a> (v0.45, for Spring 2019)</li>
  <li>Right click the downloaded zip file and choose “extract here” or “extract all”. A new folder will appear called “kallisto”. Move it to your “Program Files” directory.</li>
  <li>Kallisto is now installed on your computer but cannot be accessed from any location in the command prompt (Windows equivalent of Terminal) until you add it to your computer’s PATH system variable</li>
  <li>Add the kallisto directory to your PATH to allow for access from any directory
    <ul>
      <li>Open the Control Panel and navigate to System and security -&gt; System -&gt; Advanced System Setting -&gt; Environmental Variables</li>
      <li>Under System Variables (<strong>not</strong> User Variables), click PATH -&gt; Edit -&gt; New
Note: You’ll see a list of folders, as this example for my system shows: C:\Program Files\Windows Resource Kits\Tools\; etc</li>
      <li>Add the path to your kallisto folder to System Variables by copying the path from File Explorer (e.g. C:/Program Files\kallisto) and pasting it into the prompt.</li>
    </ul>
  </li>
  <li>Relaunch command prompt and check for proper installation by typing ‘kallisto’ (without quotes).</li>
  <li>Although not required for running kallisto, you should install <a href="https://www.cygwin.com/">Cygwin</a> to give your PC some linux functionality (like running shell scripts)</li>
</ol>

<h2 id="build-an-index-from-reference-transcriptome-fasta-file">Build an index from reference transcriptome .fasta file</h2>

<div class="alert alert-info" role="alert"><i class="fa fa-info-circle"></i> <b>Note:</b> If you are working on the PennVet CHMI linux cluster, we have prebuilt kallisto indicies from mouse, human and several other species located in /data/reference_db/kallisto</div>

<p>Get reference transcriptome files from <a href="http://useast.ensembl.org/info/data/ftp/index.html">here</a>
Search for your organism, select cDNA, then download the file that ends in “cDNA.all.fa.gz”</p>

<p>Build the index</p>

<div class="language-shell highlighter-rouge"><pre class="highlight"><code>kallisto index -i inputFastaName.index inputFastaName.fasta
</code></pre>
</div>

<div class="alert alert-info" role="alert"><i class="fa fa-info-circle"></i> <b>Note:</b> It’s a good idea to name you index exactly the same as your input fasta file, just replacing .fasta or .fa, with .index.  This naming convention makes it clear to others (and your future self) exactly which reference transcriptome was used for mapping.</div>

<h2 id="align-single-end-reads">align single-end reads</h2>

<p>Run the following command for pseudoalignment of single-end reads to index.</p>

<div class="language-shell highlighter-rouge"><pre class="highlight"><code>kallisto quant -i inputFastaName.index -o sample1_kallisto -b 60 —-single -l 275 -s 30 sample1_read1.fastq.gz
</code></pre>
</div>
<p>Once read mapping is complete, you will see a short report printed to your screen that indicates the number of reads kallisto saw in the fastq file, and the number of these that mapped to the reference.  Often times it is useful to automatically store this information in a log file so that it can be parsed by other programs, such as the incredibly useful <a href="http://multiqc.info/">multiQC</a>.  To do this, simply append the code below at the end of your alignment bit above. The outcome will be the same, but instead of displaying on the screen it will be piped to a log file.</p>

<div class="language-shell highlighter-rouge"><pre class="highlight"><code><span class="gp">&amp;&gt; </span>sample1_kallisto.log
</code></pre>
</div>

<div class="alert alert-danger" role="alert"><i class="fa fa-exclamation-circle"></i> <b>Warning:</b> avoid putting hyphens in the name of the kallisto output, as this could cause problems later</div>

<div class="alert alert-warning" role="alert"><i class="fa fa-warning"></i> <b>Important:</b> bootstrapping (-b command in the line above) adds significant time to the mapping, but is essential for accurate quantification. With a typical reference index for the mouse or human transcriptome, I find it takes about 15sec per bootstrap. So expect this to add ~30 min to the mapping time for each sample.</div>

<div class="alert alert-warning" role="alert"><i class="fa fa-warning"></i> <b>Important:</b> If you want a .bam file output with your alignment, use <code class="highlighter-rouge">--pseudobam</code>.</div>

<div class="alert alert-warning" role="alert"><i class="fa fa-warning"></i> <b>Important:</b> If you have multiple cores on your computer, you can speed up the aligment with the <code class="highlighter-rouge">-t</code> argument followed by the number of threads you have on your machine.  If you’re on a mac, you can find out the number of virtual cores (i.e. threads) using the <code class="highlighter-rouge">sysctl hw.ncpu</code> command directly in the terminal</div>

<h2 id="align-paired-end-reads">align paired-end reads</h2>

<div class="language-shell highlighter-rouge"><pre class="highlight"><code>kallisto quant -i inputFastaName.index -o sample1_kallisto -b 100 sample1_read1.fastq.gz sample1_read2.fastq.gz
</code></pre>
</div>

<h2 id="stranded-alignments-and-bigwigs">stranded alignments and bigwigs</h2>
<p>In some cases, you may want to carry out a stranded alignment with the end goal of viewing read ‘pileups’ on a genome browser track.  This can also be done using Kallisto, but requires a few other programs and steps to get from the Kallisto alignment to bigwig.</p>

<div class="alert alert-danger" role="alert"><i class="fa fa-exclamation-circle"></i> <b>Warning:</b> If you’ve been using your laptop thus far, now would be a good time to consider finding some more compute resources. Working with .bam files may be too much for your laptop to handle.</div>

<p>Stranded alignment using pseudobam. SAM creation is also possible at this step.</p>

<div class="language-shell highlighter-rouge"><pre class="highlight"><code>kallisto quant -i <span class="o">[</span>yourindex] -o <span class="o">[</span>outputname] --fr-stranded -b 60 --pseudobam <span class="o">[</span>input1] <span class="o">[</span>input2] | samtools view -Sb - &gt; <span class="o">[</span>kallisto.fr.bam]
kallisto quant -i <span class="o">[</span>yourindex] -o <span class="o">[</span>outputname] --rf-stranded -b 60 --pseudobam <span class="o">[</span>input1] <span class="o">[</span>input2] | samtools view -Sb - &gt; <span class="o">[</span>kallisto.rf.bam]
</code></pre>
</div>

<p>Sort and index using <a href="http://samtools.sourceforge.net/">samtools</a></p>

<div class="language-shell highlighter-rouge"><pre class="highlight"><code>samtools sort -@ 24 <span class="o">[</span>kallisto.fr.bam] <span class="o">[</span>kallisto.fr.sorted]
samtools sort -@ 24 <span class="o">[</span>kallisto.rf.bam] <span class="o">[</span>kallisto.rf.sorted]
samtools index <span class="o">[</span>kallisto.fr.sorted.bam]
samtools index <span class="o">[</span>kallisto.fr.sorted.bam]
</code></pre>
</div>

<p>BAM to BigWig conversion using <a href="https://deeptools.readthedocs.io/en/latest/">deeptools</a></p>

<div class="language-shell highlighter-rouge"><pre class="highlight"><code>bamCoverage –b <span class="o">[</span>kallisto.fr.sorted.bam] –o <span class="o">[</span>kallisto.fr.sorted.bw] -p max
bamCoverage –b <span class="o">[</span>kallisto.rf.sorted.bam] –o <span class="o">[</span>kallisto.rf.sorted.bw] -p max
</code></pre>
</div>

<h2 id="install-sleuth">install Sleuth</h2>
<p>Open RStudio and install the <a href="http://bioconductor.org/packages/release/bioc/html/rhdf5.html">rhdf5 package</a> from the BioC website</p>

<p>install the <a href="https://cran.r-project.org/web/packages/devtools/README.html">devtools package</a> (if you don’t already have it)</p>

<p>install the <a href="https://github.com/pachterlab/sleuth">Sleuth package</a> directly from Lior Pachter’s github page using:</p>

<div class="language-shell highlighter-rouge"><pre class="highlight"><code>devtools::install_github<span class="o">(</span><span class="s2">"pachterlab/sleuth"</span><span class="o">)</span>
</code></pre>
</div>

<h2 id="using-sleuth">Using Sleuth</h2>

<ul>
  <li>open Rstudio and load the Sleuth package <code class="highlighter-rouge">library(sleuth)</code></li>
  <li>read in a study design file with column called ‘path’ which lists the path to each Kallisto output folder</li>
  <li>set-up a study design using the model.matrix function in R, with an intercept: <code class="highlighter-rouge">myDesign &lt;- model.matrix(~treatment)</code></li>
</ul>



    <div class="tags">
        
        <b>Tags: </b>
        
        
        
        <a href="tag_bioinformatics.html" class="btn btn-default navbar-btn cursorNorm" role="button">bioinformatics</a>
        
        
        
        <a href="tag_transcriptomics.html" class="btn btn-default navbar-btn cursorNorm" role="button">transcriptomics</a>
        
        
        
    </div>



    <div id="disqus_thread"></div>
    <script type="text/javascript">
        /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
        var disqus_shortname = 'http-hostmicrobe-org'; // required: replace example with your forum shortname

        /* * * DON'T EDIT BELOW THIS LINE * * */
        (function() {
            var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
            dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
            (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
        })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>


</div>

<hr class="shaded"/>

<footer>
            <div class="row">
                <div class="col-lg-12 footer">
               &copy;2020 PennVet Center for Host-Microbial Interactions. All rights reserved. <br />
 Site last generated: Mar 23, 2020 <br />
<p><img src="images/CHMI_logo.png" alt="Company logo"/></p>
                </div>
            </div>
</footer>


        </div>
    <!-- /.row -->
</div>
<!-- /.container -->
    </div>

</body>

<!-- the google_analytics_id gets auto inserted from the config file -->



<script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','//www.google-analytics.com/analytics.js','ga');ga('create','UA-131349832-1','auto');ga('require','displayfeatures');ga('send','pageview');</script>


</html>
