<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="description" content="post-encapsulation steps for preparing sequence-ready libraries for scRNAseq">
<meta name="keywords" content="Library preptranscriptomics,  transcriptomics, library prep">
<title>Single Cell RNAseq – library prep | CHMI services</title>
<link rel="stylesheet" href="css/syntax.css">


<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
<!--<link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">-->
<link rel="stylesheet" href="css/modern-business.css">
<link rel="stylesheet" href="css/lavish-bootstrap.css">
<link rel="stylesheet" href="css/customstyles.css">
<link rel="stylesheet" href="css/theme-blue.css">

<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-cookie/1.4.1/jquery.cookie.min.js"></script>
<script src="js/jquery.navgoco.min.js"></script>


<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/2.0.0/anchor.min.js"></script>
<script src="js/toc.js"></script>
<script src="js/customscripts.js"></script>

<link rel="shortcut icon" href="images/favicon.ico">

<!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
<script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
<![endif]-->

<link rel="alternate" type="application/rss+xml" title="" href="http://localhost:4000/feed.xml">

    <script>
        $(document).ready(function() {
            // Initialize navgoco with default options
            $("#mysidebar").navgoco({
                caretHtml: '',
                accordion: true,
                openClass: 'active', // open
                save: false, // leave false or nav highlighting doesn't work right
                cookie: {
                    name: 'navgoco',
                    expires: false,
                    path: '/'
                },
                slide: {
                    duration: 400,
                    easing: 'swing'
                }
            });

            $("#collapseAll").click(function(e) {
                e.preventDefault();
                $("#mysidebar").navgoco('toggle', false);
            });

            $("#expandAll").click(function(e) {
                e.preventDefault();
                $("#mysidebar").navgoco('toggle', true);
            });

        });

    </script>
    <script>
        $(function () {
            $('[data-toggle="tooltip"]').tooltip()
        })
    </script>
    <script>
        $(document).ready(function() {
            $("#tg-sb-link").click(function() {
                $("#tg-sb-sidebar").toggle();
                $("#tg-sb-content").toggleClass('col-md-9');
                $("#tg-sb-content").toggleClass('col-md-12');
                $("#tg-sb-icon").toggleClass('fa-toggle-on');
                $("#tg-sb-icon").toggleClass('fa-toggle-off');
            });
        });
    </script>
    

</head>
<body>
<!-- Navigation -->
<nav class="navbar navbar-inverse navbar-fixed-top">
    <div class="container topnavlinks">
        <div class="navbar-header">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a class="fa fa-home fa-lg navbar-brand" href="index.html">&nbsp;<span class="projectTitle"> Center for Host-Microbial Interactions</span></a>
        </div>
        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
            <ul class="nav navbar-nav navbar-right">
                <!-- toggle sidebar button -->
                <li><a id="tg-sb-link" href="#"><i id="tg-sb-icon" class="fa fa-toggle-on"></i> Nav</a></li>
                <!-- entries without drop-downs appear here -->
                
                
                
                <!-- entries with drop-downs appear here -->
                <!-- conditional logic to control which topnav appears for the audience defined in the configuration file.-->
                
                
                
                
			<li>



  <a class="email" title="Submit feedback" href="#" onclick="javascript:window.location='mailto:beiting@upenn.edu?subject=CHMI services feedback&body=I have some feedback about the Single Cell RNAseq – library prep page: ' + window.location.href;"><i class="fa fa-envelope-o"></i> Feedback</a>

</li>

		
                <!--comment out this block if you want to hide search-->
                <li>
                    <!--start search-->
                    <div id="search-demo-container">
                        <input type="text" id="search-input" placeholder="search...">
                        <ul id="results-container"></ul>
                    </div>
                    <script src="js/jekyll-search.js" type="text/javascript"></script>
                    <script type="text/javascript">
                            SimpleJekyllSearch.init({
                                searchInput: document.getElementById('search-input'),
                                resultsContainer: document.getElementById('results-container'),
                                dataSource: 'search.json',
                                searchResultTemplate: '<li><a href="{url}" title="Single Cell RNAseq – library prep">{title}</a></li>',
                    noResultsText: 'No results found.',
                            limit: 10,
                            fuzzy: true,
                    })
                    </script>
                    <!--end search-->
                </li>
            </ul>
        </div>
        </div>
        <!-- /.container -->
</nav>

<!-- Page Content -->
<div class="container">
    <div class="col-lg-12">&nbsp;</div>
    <!-- Content Row -->
    <div class="row">
        
        
            <!-- Sidebar Column -->
            <div class="col-md-3" id="tg-sb-sidebar">
                












<ul id="mysidebar" class="nav">
    <li class="sidebarTitle">CHMI services </li>
    
    
    
        
    
    <li>
        <a href="#">Overview</a>
        <ul>
            
            
            
            <li><a href="index.html">Welcome</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">RNAseq</a>
        <ul>
            
            
            
            <li><a href="mydoc_RNAseq_FAQs.html">RNAseq FAQs</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_TruSeq.html">mRNA library prep</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_total_TruSeq.html">Total transcriptome library prep</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_lowInput_RNAseq.html">Low input protocol</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_scRNAseq.html">Single cell sequencing</a></li>
            
            
            
            <li class="subfolders">
                <a href="#">Data analysis</a>
                <ul>
                    
                    
                    
                    <li><a href="mydoc_kallisto.html">Kallisto and Sleuth</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_RNAseq_depositToGEO.html">Submitting to GEO</a></li>
                    
                    
                    
                </ul>
            </li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Microbiology</a>
        <ul>
            
            
            
            <li><a href="mydoc_micro_FAQs.html">Microbiology FAQs</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_16S.html">16S rRNA gene sequencing</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_NexteraXT.html">Shotgun metagenomics</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_16S_Sanger.html">Sanger sequencing of 16S</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_mock.html">Culture collection</a></li>
            
            
            
            <li class="subfolders">
                <a href="#">Data analysis</a>
                <ul>
                    
                    
                    
                    <li><a href="mydoc_qiime2.html">QIIME2</a></li>
                    
                    
                    
                </ul>
            </li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Compute resources</a>
        <ul>
            
            
            
            <li><a href="mydoc_linux.html">Linux cluster</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_software.html">Available software</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_RStudio.html">R Studio Server</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Cell culture</a>
        <ul>
            
            
            
            <li><a href="mydoc_3Dculture.html">3D intestinal cultures</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">In the field</a>
        <ul>
            
            
            
            <li><a href="mydoc_nanopore.html">Oxford Nanopore</a></li>
            
            
            
            <li class="subfolders">
                <a href="#">Biomeme portable QPCR</a>
                <ul>
                    
                    
                    
                    <li><a href="mydoc_biomeme.html">Set-up</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_biomeme_avian.html">Avian</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_biomeme_human.html">Human</a></li>
                    
                    
                    
                </ul>
            </li>
            
            
            
            
        </ul>
     </li>
       
        
        
        <!-- if you aren't using the accordion, uncomment this block:
           <p class="external">
               <a href="#" id="collapseAll">Collapse All</a> | <a href="#" id="expandAll">Expand All</a>
           </p>
           -->
</ul>

<!-- this highlights the active parent class in the navgoco sidebar. this is critical so that the parent expands when you're viewing a page. This must appear below the sidebar code above. Otherwise, if placed inside customscripts.js, the script runs before the sidebar code runs and the class never gets inserted.-->
<script>$("li.active").parents('li').toggleClass("active");</script>

            </div>
            
        

        <!-- Content Column -->
        <div class="col-md-9" id="tg-sb-content">
            <div class="post-header">
   <h1 class="post-title-main">Single Cell RNAseq – library prep</h1>
</div>



<div class="post-content">

   
    <div class="summary">post-encapsulation steps for preparing sequence-ready libraries for scRNAseq</div>
   

    
    
<!-- this handles the automatic toc. use ## for subheads to auto-generate the on-page minitoc. if you use html tags, you must supply an ID for the heading element in order for it to appear in the minitoc. -->
<script>
$( document ).ready(function() {
  // Handler for .ready() called.

$('#toc').toc({ minimumHeaders: 0, listType: 'ul', showSpeed: 0, headers: 'h2,h3,h4' });

/* this offset helps account for the space taken up by the floating toolbar. */
$('#toc').on('click', 'a', function() {
  var target = $(this.getAttribute('href'))
    , scroll_target = target.offset().top

  $(window).scrollTop(scroll_target - 10);
  return false
})
  
});
</script>

<div id="toc"></div>

    


    

    <a target="_blank" href="https://github.com/CHMI-SOPs/CHMI-SOPs.github.io/tree/master/pages/mydoc/mydoc_scRNAseq_libraryPrep.md" class="btn btn-default githubEditButton" role="button"><i class="fa fa-github fa-lg"></i> Edit me</a>

    

  <h1 id="overview">Overview:</h1>

<p>Day 1 - thaw samples, enzymatic digestion to remove the unwanted nucleic acids and primer dimers, then second strand synthesis. Linear amplification of the dsDNA made during second stand synthesis that takes place over night (between 13-15 hour incubation).</p>

<p>Day 2 - Check library on tapestation/bioanalyzer, amplified RNA is fragmented and used for second reverse transcription reaction to convert sample back to DNA and to add a read primer binding site to each molecule. Clean up with Ampure XP beads, amplified by PCR. qPCR is performed to determine optimal amount of cycles for the final PCR; once final PCR is finished, library can be QCed by tapestation and Qubit.</p>

<p>Bead clean ups are performed after every enzymatic step</p>

<div class="alert alert-danger" role="alert"><i class="fa fa-exclamation-circle"></i> <b>Warning:</b> AMpure XP beads need to be brought to room temperature for at least 30 minutes prior to use. Vortex beads well for at least 2 minutes before use.</div>

<p>Full protocol can be found here: https://1cell-bio.com/wp-content/uploads/2017/03/InDrop_LibraryPrep_Protocol_v1.2.pdf</p>

<h2 id="reagents-needed">Reagents needed:</h2>
<ul>
  <li><a href="https://www.thermofisher.com/order/catalog/product/10977023">Nuclease-free water, Life Technologies, cat. no. 10977-023</a></li>
  <li><a href="https://www.fishersci.com/shop/products/ampure-xp-60ml/nc9933872">Agencourt AMPure XP magnetic beads, Beckman Coulter, cat. no. A63881</a></li>
  <li><a href="https://www.neb.com/products/m0293-exonuclease-i-e-coli#Product%20Information">Exonuclease I, ExoI; 20,000 U/mL; NEB, cat. no. M0293L</a></li>
  <li><a href="https://www.thermofisher.com/order/catalog/product/FD0804">FastDigest HinFI restriction endonuclease, HinFI; Thermo Fisher/Life Technologies, cat. no. FD0804</a></li>
  <li><a href="https://www.thermofisher.com/order/catalog/product/B64?SID=srch-hj-B64">FastDigest Buffer 10x, ThermoFisher/Life Technologies, cat. no. B64</a></li>
  <li><a href="https://www.neb.com/products/e6111-nebnext-ultra-ii-non-directional-rna-second-strand-synthesis-module#Product%20Information">NEBNext® mRNA Second Strand Synthesis Module, NEB, cat. no. E6111S</a></li>
  <li><a href="https://www.neb.com/products/e2040-hiscribe-t7-high-yield-rna-synthesis-kit#Product%20Information">HiScribeTM T7 High Yield RNA Synthesis Kit, NEB, cat. no. E2040S</a></li>
  <li><a href="https://www.thermofisher.com/order/catalog/product/AM8740">RNA Fragmentation Reagents, Ambion/Life technologies, cat. no. AM8740</a></li>
  <li><a href="https://www.takarabio.com/products/cdna-synthesis/reverse-transcriptases/primescript?catalog=2680B">PrimeScriptTM Reverse Transcriptase, Takara Clontec, cat. no. 2680A</a></li>
  <li><a href="https://www.thermofisher.com/order/catalog/product/10777019">RNaseOUT Recombinant Ribonuclease Inhibitor, RNaseOUT,ThermoFisher Scientific, cat. no. 10777-019</a></li>
  <li><a href="https://www.neb.com/products/n0447-deoxynucleotide-dntp-solution-mix#Product%20Information">Deoxynucleotide (dNTP) Solution Mix, 10 mM each; NEB, cat. no. N0447L</a></li>
  <li><a href="https://www.kapabiosystems.com/product-applications/products/pcr-2/kapa-hifi-pcr-kits/#accordion-order">Kapa 2x HiFi Hot Start PCR mix, Kapa Biosystems, cat. no. KK2601</a></li>
  <li><a href="https://biotium.com/product/evagreen-dye-20x-in-water/">Eva Green Dye, 20x in water; Biotium, cat. no. 31000-T</a></li>
  <li>100 μM PE2-N6 primer (1Cell)</li>
  <li>5 μM PE1/PE2 primer mixture with indices (1Cell)</li>
  <li>100 μM Custom read 1, read2, and index primers (1Cell)</li>
</ul>

<h2 id="consumables-needed">Consumables needed:</h2>
<ul>
  <li><a href="https://www.sigmaaldrich.com/catalog/product/sigma/cls8162?lang=en&amp;region=US">Corning Costar Spin-X tube filters, Sigma-Aldrich, cat. no. CLS8162</a></li>
  <li>Bioanalyzer RNA and HSDNA chips and reagents <em>or</em> Agilent TapeStation HSRNA and HSD1000screen tapes and reagents (Aglient)</li>
</ul>

<h2 id="equipment-needed">Equipment needed:</h2>
<ul>
  <li>Magnetic racks for PCR and/or 1.5 ml tubes</li>
  <li>Centrifuge (e.g., Eppendorf AG, Cat. No. 5424R)</li>
  <li>Thermo-conductive tube rack (VWR Scientific Cat. No. 95045-464)</li>
  <li>Thermal cycler</li>
  <li>qPCR machine</li>
  <li>Thermomixer or dry bath for 1.5 ml tubes</li>
  <li>BioAnalyzer (Agilent) or Tapestation (Agilent)</li>
</ul>

<h2 id="make-these-before-you-start">Make these before you start:</h2>

<p><strong>RNA Elution Buffer (RE)</strong>:
Combine the follow in a 15 mL Falcon Tube:
- 150 uL of 1 M Tris-HCl pH 7.5
- 3 uL of 0.5 M EDTA
- Add nuclease-free water to bring the final volume to 15 mL
* This can be aliquoted into nuclease-free tubes and aliquots can be stored at 4C short term or -20C long term.</p>

<p><strong>DNA Suspension Buffer (DS)</strong>:
-  150 uL of 1 M Tris-HCl pH 8.0
- 3 uL of 0.5 M EDTA
- Add nuclease-free water to bring the final volume to 15 mL
* This can be aliquoted into nuclease-free tubes and aliquots can be stored at 4C short term or -20C long term.</p>

<p><strong>Blocking Buffer</strong> (used to block the pipet tips so your solution will not stick to the outside or the inside of your tip, block tip for all steps that you are pipetting just your sample):</p>

<p>Combine the follow in a 15 mL Falcon Tube:
- 100 uL of 1 M Tris-HCl pH 8.0
- 2 uL of 0.5 M EDTA
- 500 uL of 10% Tween 20 (vol/vol) 
- 500 uL of 2% BSA (wt/vol)
- 8898 uL of Sterile Water
Total volume is 10 mL, aliquot into 1.5 mL tubes with approximately 0.5 mL per tube, store at -20C and thaw before library prep</p>

<div class="alert alert-info" role="alert"><i class="fa fa-info-circle"></i> <b>Note:</b> Do not reuse an aliquot of blocking buffer once it was thawed for the day. Take out an aliquot a day and discard after use..</div>

<p><strong>80% Ethanol</strong>
- Make fresh each day for Ampure XP bead clean ups</p>

<h1 id="library-prep-procedure">Library Prep Procedure</h1>
<p>## Day 1</p>

<p><strong>Enzymatic Digestion</strong></p>

<p>Time: 2 hours 
1. Thaw post-RT libaries on ice
2. Centrifuge at 4C for 5 mins at 19,000 rcf to pellet any cell debris in the tube, this will bring down any material more dense than your samples.
3. Prepare the Digestion Mix. For every 70uL of post-RT material, prepare 100uL of Digestion Mix (1000 cells is approximately equal to 35uL of post-RT material so if you sorted 3000 cells, make 150uL of Digestion Mix)</p>

<p><strong>Volumes for 50uL of Digestion Mix (1000 cells):</strong>
| Component | Volume/ rxn |
|——-|——–|
| ddH2O | 39.5 uL |
| 10X Fast Digest Buffer | 4.5 uL |
| Exonuclease I | 2.5 uL |
| HinFI | 3.5 uL |
|  | Total 50 uL |</p>

<p><strong>Volumes for 100uL of Digestion Mix (2000 cells):</strong>
| Component | Volume/ rxn |
|——-|——–|
| ddH2O | 79 uL |
| 10X Fast Digest Buffer | 9 uL |
| Exonuclease I | 5 uL |
| HinFI | 7 uL |
|  | Total 100 uL |</p>

<p><strong>Volumes for 150uL of Digestion Mix (3000 cells):</strong>
| Component | Volume/ rxn |
|——-|——–|
| ddH2O | 118.5 uL |
| 10X Fast Digest Buffer | 13.5 uL |
| Exonuclease I | 7.5 uL |
| HinFI | 10.5 uL |
|  | Total 150 uL |</p>

<div class="alert alert-warning" role="alert"><i class="fa fa-warning"></i> <b>Important:</b> All steps before linear amplification before in vitro transcription should be performed with a blocked tip. This miminizes the material loss due to adsorption. Immerse pipet tip into block buffer, pipet up and down once and the tip is ready for use..</div>

<ol>
  <li>
    <p>Without aspirating HFE-7500 oil from the bottom of the tube, carefully transfer the post-RT material from each saple to its own Costar Spin-X tube filter with a blocked gel loading tip.</p>
  </li>
  <li>
    <p>Add Digestion mix made in previous step to the top of your old tube with the HFE-7500 oil to wash off any residual sample that may still be on the top of the oil. Then transfer the digestion buffer mix to the Costar-x spin tube.</p>
  </li>
  <li>
    <p>Centrifuge the filter column at 4C for 5 mins at &gt;5000 xg.</p>
  </li>
  <li>
    <p>Discard the filter membrane (this contains the beads and all the particles we wanted to filter out, your sample is in the supernatant in the flow through).</p>
  </li>
  <li>
    <p>Purify the reaction by using AmpPure XP beads. At this stage, your products are RNA-cDNA hybrids. This is the general protocol for all AmpPure XP bead clean ups. Pay attention to the ratio in each step of beads: sample, the elution buffer used and the amount of elution buffer in each clean up. These will be different in the subsequent steps.</p>
  </li>
</ol>

<ul>
  <li>
    <p>Ratio of AMpure XP beads to sample is 1.2X ; The elution buffer used here is <strong>nuclease free water</strong> at a volume of 17 uL</p>
  </li>
  <li>Add volume of beads directly to the sample. Vortex to mix well.</li>
  <li>Incubate the sample at room temperature for 5 minutes. Your sample will bind to the beads in this step.</li>
  <li>Briefly spin down tube.</li>
  <li>Place the sample(s) on the magnetic stand for 3-5 minutes until the bead collects on the side of the well and the supernatant is clear.</li>
  <li>Aspirate and discard the supernatant without disturbing the beads.</li>
  <li>Keep the sample(s) on the magent and fill with 80% etoh making sure the bead is completely covered. Remove etoh from sample after 30 seconds.</li>
  <li>Repeat wash step once more.</li>
  <li>After the second wash, spin tube down to get any excess etoh off the bead, aspirate out and discard.</li>
  <li>Dry tubes at 37C with caps off for about 3 minutes or until the beads are dry. They should appear matte and begin to crack when dry.</li>
  <li>Add 17uL of elutent and vortex, spin down, and incubate at room temperature for 5 minutes.</li>
  <li>Return the tubes to the magnetic stand and incubate for 1 minute until supernatant is clear. supernatant now contains your sample.</li>
  <li>Transfer the supernatant to a fresh tube, a PCR tube.</li>
</ul>

<p><strong>Second Strand Synthesis and In Vitro-Translation (IVT)</strong></p>

<p>Time: 16-18 hours</p>

<ol>
  <li>Set up Second Strand Synthesis (SSS) on ice:
    <ul>
      <li>Add 2 uL of 10x SSS buffer to the 17 uL eluate from above in the PCR tube.</li>
      <li>Mix by vortexing, spin down. Add 1 uL of SSS enzyme and mix gently by pipetting up and down.</li>
      <li>Incubate the reaction at 16C for 2.5 hours, then 65C for 20 minutes.</li>
    </ul>
  </li>
  <li>Make a master mix of the following components by multiplying the volume by the amount of samples you have; add 60 uL to each sample:</li>
</ol>

<table>
  <thead>
    <tr>
      <th>Component</th>
      <th>Volume/ rxn</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Nuclease-free water</td>
      <td>12 uL</td>
    </tr>
    <tr>
      <td>NEB T7 High Yield 10X Buffer</td>
      <td>8 uL</td>
    </tr>
    <tr>
      <td>NTP [4 different NTP buffers, ATP, UTP, CTP, GTP] (100mM)</td>
      <td>8 uL each</td>
    </tr>
    <tr>
      <td>NEB T7 High Yield Enzyme Mix</td>
      <td>8 uL</td>
    </tr>
    <tr>
      <td> </td>
      <td>Total 60 uL</td>
    </tr>
  </tbody>
</table>

<ol>
  <li>Pipette to mix, spin plate down, and incubate at 37C for 13-15 hours with the lid heated to 50C.</li>
</ol>

<ul>
  <li><strong>Pause point</strong> After 15h the IVT reaction can be kept at 4C for two hours, this is programmed into our PCR protocol on the machine.</li>
</ul>

<ol>
  <li>Purify the reaction by using AmpPure XP beads. This is the general protocol for all AmpPure XP bead clean ups. Pay attention to the ratio in each step of beads: sample, the elution buffer used and the amount of elution buffer in each clean up. These will be different in the subsequent steps.</li>
</ol>

<ul>
  <li>
    <p>Ratio of AMpure XP beads to sample is 1.3X (104uL of beads) ; The elution buffer used here is <strong>RNA Elution Buffer</strong> at a volume of 21 uL</p>
  </li>
  <li>Add volume of beads directly to the sample. Vortex to mix well.</li>
  <li>Incubate the sample at room temperature for 5 minutes. Your sample will bind to the beads in this step.</li>
  <li>Briefly spin down tube.</li>
  <li>Place the sample(s) on the magnetic stand for 3-5 minutes until the bead collects on the side of the well and the supernatant is clear.</li>
  <li>Aspirate and discard the supernatant without disturbing the beads.</li>
  <li>Keep the sample(s) on the magent and fill with 80% etoh making sure the bead is completely covered. Remove etoh from sample after 30 seconds.</li>
  <li>Repeat wash step once more.</li>
  <li>After the second wash, spin tube down to get any excess etoh off the bead, aspirate out and discard.</li>
  <li>Dry tubes at 37C with caps off for about 3 minutes or until the beads are dry. They should appear matte and begin to crack when dry.</li>
  <li>Add 21uL of elutent and pipette to mix, spin down, and incubate at room temperature for 5 minutes.</li>
  <li>Return the tubes to the magnetic stand and incubate for 1 minute until supernatant is clear. supernatant now contains your sample.</li>
  <li>Transfer 9uL of supernatant to a new PCR tube. Transfer 10uL to a LoBind tube. This can be stored in the -80C for up to 6 months, labeled as post-IVT back up.</li>
</ul>

<ol>
  <li>Run a HS RNA Tapestation on the post-IVT back up before transferring to the -80C freezer.</li>
</ol>

<p><strong>Fragmentation of Amplified RNA</strong></p>

<p>Time: 20 mins</p>

<ol>
  <li>Prepare Fragmentation Stop Mix for each sample:</li>
</ol>

<table>
  <thead>
    <tr>
      <th>Component</th>
      <th>Volume/ rxn</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>RNA Elution Buffer</td>
      <td>9.8 uL</td>
    </tr>
    <tr>
      <td>AMPure Beads</td>
      <td>26.4 uL</td>
    </tr>
    <tr>
      <td>Stop Solution from the RNA fragementation kit</td>
      <td>1.2uL</td>
    </tr>
    <tr>
      <td> </td>
      <td>Total  37.4 uL</td>
    </tr>
  </tbody>
</table>

<ol>
  <li>
    <p>Vortex the solution from above and place on ice.</p>
  </li>
  <li>
    <p>Allow the thermalcycler to reach 70C (lid at 100-110C)</p>
  </li>
  <li>
    <p>On ice, add 1 uL of 10X RNA Fragmentation reagent to 9 uL of purified of IVT product in plate from step 12.</p>
  </li>
  <li>
    <p>Spin down plate and fragment on thermalcycler at 70C for about 1-3 minutes, depending on what your post IVT tapestation looked like. Here is how to determine the fragmentation time:</p>
  </li>
</ol>

<ul>
  <li>1cellbio’s general rule is fragment for around 1 min if the bulk of the sample is below 1000nt, 2 mins if the bulk of the sample is between 1000-2000nt, and 3 mins if the majority of your sample is above 2000nt.</li>
</ul>

<ol>
  <li>
    <p>Cool the fragmentation reaction on ice and immediately add 34uL of Fragmentation Stop Mix (prepared in step 14) to each sample and pipette up and down to mix well. Spin plate very briefly.</p>
  </li>
  <li>
    <p>Incubate samples on ice for 5 minutes.</p>
  </li>
  <li>
    <p>Plate samples on magnetic rack and continue through the AMPure XP bead clean up steps below:</p>
  </li>
</ol>

<p>Ratio of AMpure XP beads to sample is 1.3X (104uL of beads) ; The elution buffer used here is <strong>RNA Elution Buffer</strong> at a volume of 21 uL</p>

<ul>
  <li>Place the sample(s) on the magnetic stand for 3-5 minutes until the bead collects on the side of the well and the supernatant is clear.</li>
  <li>Aspirate and discard the supernatant without disturbing the beads.</li>
  <li>Keep the sample(s) on the magent and fill with 80% etoh making sure the bead is completely covered. Remove etoh from sample after 30 seconds.</li>
  <li>Repeat wash step once more.</li>
  <li>After the second wash, spin tube down to get any excess etoh off the bead, aspirate out and discard.</li>
  <li>Dry tubes or plate at 37C with caps off for about 3 minutes or until the beads are dry. They should appear matte and begin to crack when dry.</li>
  <li>Add 10uL of elutent and pipette to mix, spin down, and incubate at room temperature for 5 minutes.</li>
  <li>Return the tubes to the magnetic stand and incubate for 1 minute until supernatant is clear. Supernatant now contains your sample.</li>
  <li>Transfer 8uL of supernatant to a new well.</li>
</ul>

<p><strong>Reverse Transcription using Random Hexamers</strong></p>

<p>Time: 2.5 hours</p>

<ol>
  <li>On ice, combine the following:</li>
</ol>

<table>
  <thead>
    <tr>
      <th>Component</th>
      <th>Volume/ rxn</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Purified Fragmented amplified RNA from previous step</td>
      <td>8 uL</td>
    </tr>
    <tr>
      <td>dNTP (10mM)</td>
      <td>1 uL</td>
    </tr>
    <tr>
      <td>PE2-N6 primer (100uM)</td>
      <td>2uL</td>
    </tr>
    <tr>
      <td> </td>
      <td>Total  11 uL</td>
    </tr>
  </tbody>
</table>

<ol>
  <li>
    <p>Pipette to mix, spin down. Incubate at 70C (lid at 105C) for exactly 3 minutes and cool immediately on ice.</p>
  </li>
  <li>
    <p>On ice, add the following components to the mix to bring the total volume to 20uL:</p>
  </li>
</ol>

<table>
  <thead>
    <tr>
      <th>Component</th>
      <th>Volume/ rxn</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Nuclease-free water</td>
      <td>3.5 uL</td>
    </tr>
    <tr>
      <td>5x Prime Script Buffer</td>
      <td>4 uL</td>
    </tr>
    <tr>
      <td>RNase OUT (40 U/uL)</td>
      <td>1 uL</td>
    </tr>
    <tr>
      <td>Prime Script Reverse Transcriptase (200 U/uL)</td>
      <td>0.5 uL</td>
    </tr>
    <tr>
      <td> </td>
      <td>Total  9 uL</td>
    </tr>
  </tbody>
</table>

<ol>
  <li>
    <p>Mix gently by pipetting and incubate the reaction at 30C for 10 minutes followed by 42C for 1 hour then 70C for 15 minutes to heat inactivate the enzyme.</p>
  </li>
  <li>
    <p>Add 20uL of nuclease-free water, mix by pipetting, and save 20 uL as “pre-PCR back-up” in a LoBind tube. This back up can be stored at -80C for up to 6 months.</p>
  </li>
  <li>
    <p>Purify the reaction with AMPure XP Beads</p>
  </li>
</ol>

<ul>
  <li>
    <p>Ratio of AMpure XP beads to sample is 1.2X (24 uL) ; The elution buffer used here is <strong>DNA elution buffer</strong> at a volume of 12 uL</p>
  </li>
  <li>Add volume of beads directly to the sample. Pipette to mix well.</li>
  <li>Incubate the sample at room temperature for 5 minutes. Your sample will bind to the beads in this step.</li>
  <li>Place the sample(s) on the magnetic stand for 3-5 minutes until the bead collects on the side of the well and the supernatant is clear.</li>
  <li>Aspirate and discard the supernatant without disturbing the beads.</li>
  <li>Keep the sample(s) on the magent and fill with 80% etoh making sure the bead is completely covered. Remove etoh from sample after 30 seconds. The ethanol should fully cover the bead in this step, in a plate add 180 uL of the 80% ethanol.</li>
  <li>Repeat wash step once more.</li>
  <li>After the second wash, use a P10 to get remaining ethanol out of sample and discard.</li>
  <li>Dry samples at 37C with no cover for about 3 minutes or until the beads are dry. They should appear matte and begin to crack when dry.</li>
  <li>Add 12uL of elutent and pipette to mix.</li>
  <li>Incubate at room temperature for 5 minutes.</li>
  <li>Return the tubes to the magnetic stand and incubate for 1 minute until supernatant is clear. Supernatant now contains your sample.</li>
  <li>Transfer 10 uL of supernatant to a new well.</li>
</ul>

<p><strong>Diagnostic qPCR</strong></p>

<p>Time: 1.5-2 hours</p>

<ol>
  <li>Perform a quantitative PCR (qPCR) to determine the correct number of PCR amplification cycles in the next step. Combine the following components on ice:</li>
</ol>

<table>
  <thead>
    <tr>
      <th>Component</th>
      <th>Volume/ rxn</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Nuclease-free water</td>
      <td>6.5 uL</td>
    </tr>
    <tr>
      <td>Eluate from step 27</td>
      <td>0.5 uL</td>
    </tr>
    <tr>
      <td>2x Kapa HiFi Hot Start PCR Mix</td>
      <td>10 uL</td>
    </tr>
    <tr>
      <td>20x Eva Green Dye</td>
      <td>1 uL</td>
    </tr>
    <tr>
      <td>PE1/PE2 primer mix (5uM)</td>
      <td>2 uL</td>
    </tr>
    <tr>
      <td> </td>
      <td>Total  20 uL</td>
    </tr>
  </tbody>
</table>

<div class="alert alert-warning" role="alert"><i class="fa fa-warning"></i> <b>Important:</b> Use primers 6 and 12 first. If only running one sample, use primer 6, located in A6 on the well of the primer plate supplied by 1cellBio. If running two samples, use primer 6 and primer 12 (A6 and A12). If running more than two samples, once primers 6 and primer 12 are used, it does not matter which primer is used.</div>

<ol>
  <li>Perform the following qPCR thermalcycling program:</li>
</ol>

<p>Eva green uses the same excitation and emission settings as SYBR Green.</p>

<p>Italics denote steps to be run for 2 cycles.</p>

<p>Bold denotes steps to be run for 24 cycles.</p>

<table>
  <thead>
    <tr>
      <th>Temp (C)</th>
      <th>Time (min:sec)</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>98</td>
      <td>2:00</td>
    </tr>
    <tr>
      <td><em>98</em></td>
      <td><em>0:20</em></td>
    </tr>
    <tr>
      <td><em>55</em></td>
      <td><em>0:30</em></td>
    </tr>
    <tr>
      <td><em>72</em></td>
      <td><em>0:40</em></td>
    </tr>
    <tr>
      <td><strong>98</strong></td>
      <td><strong>0:20</strong></td>
    </tr>
    <tr>
      <td><strong>65</strong></td>
      <td><strong>0:30</strong></td>
    </tr>
    <tr>
      <td><strong>72</strong></td>
      <td><strong>0:40</strong> with fluorescent read</td>
    </tr>
  </tbody>
</table>

<ol>
  <li>Set the qPCR threshold within the exponential phase of amplification, but closer to where the signal starts to emerge from the noise to avoid the possibilty of over-amplification.
    <ul>
      <li>To account for the differences of input material between the qPCR and the PCR amplification of our samples, we will calculate the amount of PCR cycles to amplify our sample with now.</li>
      <li>The library input is 19 times higher than the qPCR input, therefore the difference corresponds to log2^19 = 4.25 cycles. For example, if the number of cycles determines by the qPCR is 2 + 12.67, the required number of PCR cycles for library amplification is 2 + (12.67 -4.25) = 2 + 8.</li>
    </ul>
  </li>
</ol>

<p><strong>Library Amplification</strong></p>

<p>Time: 2 hours</p>

<ol>
  <li>Combine the following components on ice:</li>
</ol>

<table>
  <thead>
    <tr>
      <th>Component</th>
      <th>Volume/ rxn</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Nuclease-free water</td>
      <td>0.5 uL</td>
    </tr>
    <tr>
      <td>Eluate from step 27</td>
      <td>9.5 uL</td>
    </tr>
    <tr>
      <td>2x Kapa HiFi Hot Start PCR Mix</td>
      <td>12.5 uL</td>
    </tr>
    <tr>
      <td>PE1/PE2 primer mix (5uM)</td>
      <td>2.5 uL</td>
    </tr>
    <tr>
      <td> </td>
      <td>Total  25 uL</td>
    </tr>
  </tbody>
</table>

<div class="alert alert-warning" role="alert"><i class="fa fa-warning"></i> <b>Important:</b> Use the same primer numbers that were used in the qPCR step. Again, use A6 and A12 first before using any other primer sets on the plate supplied by 1cellbio.</div>

<div class="alert alert-warning" role="alert"><i class="fa fa-warning"></i> <b>Important:</b> Every sample needs to have a unique primer set from this plate because this is how the samples will be determined from one another after sequencing is finished.</div>

<ol>
  <li>Perform thermal cycling protocol shown in step 29, also shown again below:</li>
</ol>

<p>Italics denote steps to be run for 2 cycles.</p>

<p>Bold denotes steps to be run for amount of cycles determined by qPCR.</p>

<table>
  <thead>
    <tr>
      <th>Temp (C)</th>
      <th>Time (min:sec)</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>98</td>
      <td>2:00</td>
    </tr>
    <tr>
      <td><em>98</em></td>
      <td><em>0:20</em></td>
    </tr>
    <tr>
      <td><em>55</em></td>
      <td><em>0:30</em></td>
    </tr>
    <tr>
      <td><em>72</em></td>
      <td><em>0:40</em></td>
    </tr>
    <tr>
      <td><strong>98</strong></td>
      <td><strong>0:20</strong></td>
    </tr>
    <tr>
      <td><strong>65</strong></td>
      <td><strong>0:30</strong></td>
    </tr>
    <tr>
      <td><strong>72</strong></td>
      <td><strong>0:40</strong></td>
    </tr>
    <tr>
      <td>72</td>
      <td>5:00</td>
    </tr>
  </tbody>
</table>

<ul>
  <li>If working with multiple libraries that require different cycle numbers, set the thermal cycler to the maximum number of cycles needed, then manually remove each individual sample when they have reached the desired cycle number.</li>
  <li>
    <p>Move the individual libraries to an ice bucket while the rest of the libraries are amplifying.</p>
  </li>
  <li>Once every library has been amplified for the correct amount of cycles, transfer all libraries back to the PCR machine for the final step, 72C for 5 minutes.</li>
</ul>

<ol>
  <li>
    <p>Add 25 uL of DNA Elution Buffer to the PCR product.</p>
  </li>
  <li>
    <p>Purify using AMPure XP beads.</p>
  </li>
</ol>

<ul>
  <li>
    <p>Ratio of AMpure XP beads to sample is 0.7X (35 uL) ; The elution buffer used here is <strong>DNA elution buffer</strong> at a volume of 22 uL</p>
  </li>
  <li>Add volume of beads directly to the sample. Pipette to mix well.</li>
  <li>Incubate the sample at room temperature for 5 minutes. Your sample will bind to the beads in this step.</li>
  <li>Place the sample(s) on the magnetic stand for 3-5 minutes until the bead collects on the side of the well and the supernatant is clear.</li>
  <li>Aspirate and discard the supernatant without disturbing the beads.</li>
  <li>Keep the sample(s) on the magent and fill with 80% etoh making sure the bead is completely covered. Remove etoh from sample after 30 seconds. The ethanol should fully cover the bead in this step, in a plate add 180 uL of the 80% ethanol.</li>
  <li>Repeat wash step once more.</li>
  <li>After the second wash, use a P10 to get remaining ethanol out of sample and discard.</li>
  <li>Dry samples at 37C with no cover for about 3 minutes or until the beads are dry. They should appear matte and begin to crack when dry.</li>
  <li>Add 22uL of elutent and pipette to mix.</li>
  <li>Incubate at room temperature for 5 minutes.</li>
  <li>Return the tubes to the magnetic stand and incubate for 1 minute until supernatant is clear. Supernatant now contains your sample.</li>
  <li>Transfer 20 uL of supernatant to a new well.</li>
</ul>

<p>This eluate is the final sequencing-ready library.</p>

<ol>
  <li>
    <p>Test library by running on the HS D1000. The library should look like a smooth hump starting at 200bp and ending at 1.5kb with an average size between 400 and 600bp.</p>
  </li>
  <li>
    <p>Perform Qubit on samples using the HS DNA assay to get the concentration for pooling and sequencing.</p>
  </li>
</ol>

<p><strong>Pause Point</strong>: store the libraries at -80C or -20C. They should be stable for at least 1 year.</p>


    <div class="tags">
        
        <b>Tags: </b>
        
        
        
        
        
        <a href="tag_transcriptomics.html" class="btn btn-default navbar-btn cursorNorm" role="button">transcriptomics</a>
        
        
        
    </div>



    <div id="disqus_thread"></div>
    <script type="text/javascript">
        /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
        var disqus_shortname = 'http-hostmicrobe-org'; // required: replace example with your forum shortname

        /* * * DON'T EDIT BELOW THIS LINE * * */
        (function() {
            var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
            dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
            (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
        })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>


</div>

<hr class="shaded"/>

<footer>
            <div class="row">
                <div class="col-lg-12 footer">
               &copy;2018 PennVet Center for Host-Microbial Interactions. All rights reserved. <br />
 Site last generated: Sep 29, 2018 <br />
<p><img src="images/CHMI_logo.png" alt="Company logo"/></p>
                </div>
            </div>
</footer>


        </div>
    <!-- /.row -->
</div>
<!-- /.container -->
    </div>

</body>

</html>
