<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="description" content="Preparation of V4 region, 16S dual-indexed libraries for sequencing on an Illumina platform">
<meta name="keywords" content="bioinformaticsmicrobiome,  ">
<title>analysis of 16S rRNA gene sequencing data | CHMI services</title>
<link rel="stylesheet" href="css/syntax.css">


<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
<!--<link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">-->
<link rel="stylesheet" href="css/modern-business.css">
<link rel="stylesheet" href="css/lavish-bootstrap.css">
<link rel="stylesheet" href="css/customstyles.css">
<link rel="stylesheet" href="css/theme-blue.css">

<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-cookie/1.4.1/jquery.cookie.min.js"></script>
<script src="js/jquery.navgoco.min.js"></script>


<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/2.0.0/anchor.min.js"></script>
<script src="js/toc.js"></script>
<script src="js/customscripts.js"></script>

<link rel="shortcut icon" href="images/favicon.ico">

<!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
<script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
<![endif]-->

<link rel="alternate" type="application/rss+xml" title="" href="http://localhost:9000/feed.xml">



    <script>
        $(document).ready(function() {
            // Initialize navgoco with default options
            $("#mysidebar").navgoco({
                caretHtml: '',
                accordion: true,
                openClass: 'active', // open
                save: false, // leave false or nav highlighting doesn't work right
                cookie: {
                    name: 'navgoco',
                    expires: false,
                    path: '/'
                },
                slide: {
                    duration: 400,
                    easing: 'swing'
                }
            });

            $("#collapseAll").click(function(e) {
                e.preventDefault();
                $("#mysidebar").navgoco('toggle', false);
            });

            $("#expandAll").click(function(e) {
                e.preventDefault();
                $("#mysidebar").navgoco('toggle', true);
            });

        });

    </script>
    <script>
        $(function () {
            $('[data-toggle="tooltip"]').tooltip()
        })
    </script>
    <script>
        $(document).ready(function() {
            $("#tg-sb-link").click(function() {
                $("#tg-sb-sidebar").toggle();
                $("#tg-sb-content").toggleClass('col-md-9');
                $("#tg-sb-content").toggleClass('col-md-12');
                $("#tg-sb-icon").toggleClass('fa-toggle-on');
                $("#tg-sb-icon").toggleClass('fa-toggle-off');
            });
        });
    </script>
    

</head>
<body>
<!-- Navigation -->
<nav class="navbar navbar-inverse navbar-fixed-top">
    <div class="container topnavlinks">
        <div class="navbar-header">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a class="fa fa-home fa-lg navbar-brand" href="index.html">&nbsp;<span class="projectTitle"> Center for Host-Microbial Interactions</span></a>
        </div>
        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
            <ul class="nav navbar-nav navbar-right">
                <!-- toggle sidebar button -->
                <li><a id="tg-sb-link" href="#"><i id="tg-sb-icon" class="fa fa-toggle-on"></i> Nav</a></li>
                <!-- entries without drop-downs appear here -->
                
                
                
                <!-- entries with drop-downs appear here -->
                <!-- conditional logic to control which topnav appears for the audience defined in the configuration file.-->
                
                
                
                
			<li>



  <a class="email" title="Submit feedback" href="#" onclick="javascript:window.location='mailto:beiting@upenn.edu?subject=CHMI services feedback&body=I have some feedback about the analysis of 16S rRNA gene sequencing data page: ' + window.location.href;"><i class="fa fa-envelope-o"></i> Feedback</a>

</li>

		
                <!--comment out this block if you want to hide search-->
                <li>
                    <!--start search-->
                    <div id="search-demo-container">
                        <input type="text" id="search-input" placeholder="search...">
                        <ul id="results-container"></ul>
                    </div>
                    <script src="js/jekyll-search.js" type="text/javascript"></script>
                    <script type="text/javascript">
                            SimpleJekyllSearch.init({
                                searchInput: document.getElementById('search-input'),
                                resultsContainer: document.getElementById('results-container'),
                                dataSource: 'search.json',
                                searchResultTemplate: '<li><a href="{url}" title="analysis of 16S rRNA gene sequencing data">{title}</a></li>',
                    noResultsText: 'No results found.',
                            limit: 10,
                            fuzzy: true,
                    })
                    </script>
                    <!--end search-->
                </li>
            </ul>
        </div>
        </div>
        <!-- /.container -->
</nav>

<!-- Page Content -->
<div class="container">
    <div class="col-lg-12">&nbsp;</div>
    <!-- Content Row -->
    <div class="row">
        
        
            <!-- Sidebar Column -->
            <div class="col-md-3" id="tg-sb-sidebar">
                












<ul id="mysidebar" class="nav">
    <li class="sidebarTitle">CHMI services </li>
    
    
    
        
    
    <li>
        <a href="#">Overview</a>
        <ul>
            
            
            
            <li><a href="index.html">Welcome</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Bulk RNAseq</a>
        <ul>
            
            
            
            <li><a href="mydoc_RNAseq_FAQs.html">FAQs</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_TruSeq.html">mRNA library prep</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_total_TruSeq.html">Total transcriptome library prep</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_lowInput_RNAseq.html">Low input protocol</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_FFPE_RNAseq.html">FFPE-seq</a></li>
            
            
            
            <li class="subfolders">
                <a href="#">RNAseq data analysis</a>
                <ul>
                    
                    
                    
                    <li><a href="mydoc_RNAseq_QA.html">Project management and QA</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_kallisto.html">Read mapping with Kallisto</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_RNAseq_depositToGEO.html">Submitting data to GEO</a></li>
                    
                    
                    
                </ul>
            </li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Single cell genomics</a>
        <ul>
            
            
            
            <li><a href="mydoc_scRNAseq_FAQs.html">scRNAseq FAQs</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_10X_prep.html">10X sample prep</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_inDrop_encapsulation.html">inDrop encapsulation</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_inDrop_libraryPrep.html">inDrop library prep</a></li>
            
            
            
            <li class="subfolders">
                <a href="#">scRNAseq data analysis</a>
                <ul>
                    
                    
                    
                    <li><a href="mydoc_CellRanger.html">10X CellRanger</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_CellRanger.html">BUStools</a></li>
                    
                    
                    
                </ul>
            </li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Microbiology</a>
        <ul>
            
            
            
            <li><a href="mydoc_micro_FAQs.html">Microbiology FAQs</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_16S.html">16S rRNA gene sequencing</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_NexteraXT.html">Shotgun metagenomics</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_16S_Sanger.html">Sanger sequencing of 16S</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_mock.html">Culture collection</a></li>
            
            
            
            <li class="subfolders">
                <a href="#">Data analysis</a>
                <ul>
                    
                    
                    
                    <li><a href="mydoc_qiime2.html">QIIME2</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_microbiomeDB_deposit.html">microbiomeDB.org</a></li>
                    
                    
                    
                    
                    
                    <li><a href="mydoc_sourmash.html">Sourmash</a></li>
                    
                    
                    
                </ul>
            </li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Compute resources</a>
        <ul>
            
            
            
            <li><a href="mydoc_linux.html">Linux cluster</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_software.html">Available software</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_RStudio.html">R Studio Server</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Cell culture</a>
        <ul>
            
            
            
            <li><a href="mydoc_3Dculture.html">3D intestinal cultures</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_3Dculture_microscopy.html">Immunofluorescence for 3D cultures</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Portable sequencing</a>
        <ul>
            
            
            
            <li><a href="mydoc_biomeme.html">ONT MinION set-up</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Portable QPCR</a>
        <ul>
            
            
            
            <li><a href="mydoc_biomeme.html">Biomeme setup</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_biomeme_avian.html">Avian</a></li>
            
            
            
            
            
            
            <li><a href="mydoc_biomeme_human.html">Cutaneous leishmaniasis</a></li>
            
            
            
            
        </ul>
     </li>
       
        
    
    <li>
        <a href="#">Workshops</a>
        <ul>
            
            
            
            <li><a href="mydoc_cloudMeta.html">Metagenomics in the cloud</a></li>
            
            
            
            
        </ul>
     </li>
       
        
        
        <!-- if you aren't using the accordion, uncomment this block:
           <p class="external">
               <a href="#" id="collapseAll">Collapse All</a> | <a href="#" id="expandAll">Expand All</a>
           </p>
           -->
</ul>

<!-- this highlights the active parent class in the navgoco sidebar. this is critical so that the parent expands when you're viewing a page. This must appear below the sidebar code above. Otherwise, if placed inside customscripts.js, the script runs before the sidebar code runs and the class never gets inserted.-->
<script>$("li.active").parents('li').toggleClass("active");</script>

            </div>
            
        

        <!-- Content Column -->
        <div class="col-md-9" id="tg-sb-content">
            <div class="post-header">
   <h1 class="post-title-main">analysis of 16S rRNA gene sequencing data</h1>
</div>



<div class="post-content">

   
    <div class="summary">Preparation of V4 region, 16S dual-indexed libraries for sequencing on an Illumina platform</div>
   

    
    
<!-- this handles the automatic toc. use ## for subheads to auto-generate the on-page minitoc. if you use html tags, you must supply an ID for the heading element in order for it to appear in the minitoc. -->
<script>
$( document ).ready(function() {
  // Handler for .ready() called.

$('#toc').toc({ minimumHeaders: 0, listType: 'ul', showSpeed: 0, headers: 'h2,h3,h4' });

/* this offset helps account for the space taken up by the floating toolbar. */
$('#toc').on('click', 'a', function() {
  var target = $(this.getAttribute('href'))
    , scroll_target = target.offset().top

  $(window).scrollTop(scroll_target - 10);
  return false
})
  
});
</script>

<div id="toc"></div>

    


    

    <a target="_blank" href="https://github.com/CHMI-SOPs/CHMI-SOPs.github.io/tree/master/pages/mydoc/mydoc_qiime.md" class="btn btn-default githubEditButton" role="button"><i class="fa fa-github fa-lg"></i> Edit me</a>

    

  <h2 id="connect-to-a-compute-cluster">Connect to a compute cluster:</h2>

<h3 id="option-1-use-pmacs">option 1: use PMACS</h3>

<p>use ssh to connect to PMACS.  You’ll be propted for your password.</p>

<div class="highlighter-rouge"><pre class="highlight"><code>ssh -X username@consign.pmacs.upenn.edu
</code></pre>
</div>

<p>Enter the interactive bash shell</p>

<div class="highlighter-rouge"><pre class="highlight"><code>bsub -Is bash
</code></pre>
</div>

<p>or for larger datasets:</p>

<div class="highlighter-rouge"><pre class="highlight"><code>bsub -Is -q denovo -n 5 bash 
</code></pre>
</div>

<p>Activate QIIME</p>

<div class="highlighter-rouge"><pre class="highlight"><code>source /opt/software/qiime/1.8.0/activate.sh
</code></pre>
</div>

<h3 id="option-2-use-chmis-linux-server">option 2: use CHMI’s linux server</h3>

<div class="highlighter-rouge"><pre class="highlight"><code>ssh username@130.91.255.137
</code></pre>
</div>

<h2 id="check-your-mapping-file">check your mapping file</h2>

<p>In order to use QIIME, you must have your mapping file correctly formatted.  An example of this format is shown in the table below.  ‘#SampleID’, ‘BarcodeSequence’, ‘LinkerPrimerSequence’ and ‘Description’ must all be present in the file and named exactly as we’ve written (case sensitive).  Specific details about your experiment (treatment, sample type, cage, etc) are added between the ‘LinkerPrimerSequence’ and ‘Description’ columns.  Here we’ve simply listed these as covariate1, covariate2, etc.  You can have as many of these columns as necessary to describe your experiment.  Note the sampleID column must be unique.</p>

<table>
  <thead>
    <tr>
      <th>#SampleID</th>
      <th>BarcodeSequence</th>
      <th>LinkerPrimerSequence</th>
      <th>covariate1</th>
      <th>covariate2</th>
      <th>covariate3</th>
      <th>Description</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>001</td>
      <td>ATCGCTACTTAGCGCT</td>
      <td>GTGCCAGCMGCCGCGGTAA</td>
      <td>cageA</td>
      <td>control</td>
      <td>wild-type</td>
      <td>WT naive mouse #1</td>
    </tr>
    <tr>
      <td>002</td>
      <td>TGCGCCACTAACCGCT</td>
      <td>GTGCCAGCMGCCGCGGTAA</td>
      <td>cageB</td>
      <td>treated</td>
      <td>wild-type</td>
      <td>WT infected mouse #1</td>
    </tr>
    <tr>
      <td>003</td>
      <td>GGCAATCACTAACCGC</td>
      <td>GTGCCAGCMGCCGCGGTAA</td>
      <td>cageA</td>
      <td>treated</td>
      <td>wild-type</td>
      <td>WT infected mouse #2</td>
    </tr>
  </tbody>
</table>

<p>Once you think you have your mapping file ready to go, run the <code class="highlighter-rouge">validate_mapping_file.py</code> line below to confirm that the format is correct.  Visualize the output html file. Yellow is ok, red is an error that needs to be fixed.</p>

<div class="highlighter-rouge"><pre class="highlight"><code>validate_mapping_file.py -m MappingFile.txt
</code></pre>
</div>

<h2 id="join-forward-and-reverse-reads">Join forward and reverse reads</h2>

<p>We begin by taking our forward and reverse reads and joining these reads.  The resulting output file (called ‘joinedReads’ in the example below) will be a single fastq file containing all of your data.</p>

<div class="highlighter-rouge"><pre class="highlight"><code>join_paired_ends.py -f myForwardReads.fastq -r myReverseReads.fastq -o JoinedReads
</code></pre>
</div>

<h2 id="dealing-with-barcodes">dealing with barcodes</h2>
<p>Our UPenn sequencing core delivers fastq files that have the barcode listed in the header.  After joining paired ends, both barcodes will be listed in the header, with a ‘+’ separating the two.  We’ll use a perl script to remove the ‘+’ in order to prevent issues later.</p>

<div class="highlighter-rouge"><pre class="highlight"><code>perl Testscript.pl JoinedReads/fastqjoin.join.fastq &gt; JoinedReads.fastq
</code></pre>
</div>

<p>With the ‘+’ removed, we then extract the barcodes and put these into new file called ‘barcodes’</p>

<div class="highlighter-rouge"><pre class="highlight"><code>extract_barcodes.py -f JoinedReads.fastq -l 16 -o Barcodes -c barcode_in_label
</code></pre>
</div>

<p>Right now, each of your sequences is associated with a specific barcode, but we haven’t yet matched these barcodes with our original sample identifiers.  This is where your mapping file comes in.  In the process of doing this mapping, we’ll also discard any sequences with <a href="https://en.wikipedia.org/wiki/Phred_quality_score">Phred</a> scores &lt; 30.</p>

<div class="highlighter-rouge"><pre class="highlight"><code>split_libraries_fastq.py -i JoinedReads.fastq -o sl_out -q 29 -b Barcodes/barcodes.fastq -m MappingFile.txt --barcode_type 16
</code></pre>
</div>

<h2 id="quality-filtering">Quality filtering</h2>

<p>We use <a href="(https://www.mothur.org/)">Mothur</a> to trim our sequences, but you can also use other programs such as the FASTX toolkit.  We’ll begin by firing up Mothur:</p>

<div class="highlighter-rouge"><pre class="highlight"><code>mothur
</code></pre>
</div>

<p>First, let’s use Mothur’s handy <code class="highlighter-rouge">summary.seqs</code> function to sumarize information about the sequence file, including binning sequences by length,  looking at the number of sequences in each bin, and finding homopolymers</p>

<div class="highlighter-rouge"><pre class="highlight"><code>summary.seqs(fasta=sl_out/seqs.fna)
</code></pre>
</div>

<p>Now, we’ll use Mothur to trim the sequences and remove homopolymer runs.  You can adjust these values based on the region you are looking at. For 16S V4 we expect sequences of 250 bp, so we trim based on that expectiation.  If you’re using another 16S region, then you may want to adjust the <code class="highlighter-rouge">minlength</code> and <code class="highlighter-rouge">maxlength</code> parameters accordingly. Once complete, we run <code class="highlighter-rouge">summary.seqs</code> again to visualize the impact of trimming on our data.  Finally, we quit mothur.</p>

<div class="highlighter-rouge"><pre class="highlight"><code>trim.seqs(fasta=sl_out/seqs.fna, maxhomop=10, minlength=248, maxlength=256)
summary.seqs(fasta=sl_out/seqs.trim.fasta)
quit()
</code></pre>
</div>

<h2 id="optional-concatenate-fasta-data-from-multiple-runs">Optional: Concatenate fasta data from multiple runs</h2>
<p>We typically put data from each run into a single large file that contains all data from all past runs.  This is something we do internally at CHMI, so if you’re running though this on your own, feel free to skip this step.</p>

<div class="highlighter-rouge"><pre class="highlight"><code>cat MiSeq1234567_seqs.fna MiSeq8_seqs.fna &gt; MiSeq12345678_seqs.fna
</code></pre>
</div>

<h2 id="generate-a-fasta-file-that-has-only-your-sequences-of-interest">Generate a fasta file that has only your sequences of interest</h2>

<p>16S microbiome sequencing runs often include 100’s of samples.  Since you many not want to analyze all the samples from a single run at the same time, you can use the QIIME <code class="highlighter-rouge">filter_fasta.py</code> function to select only samples of interest.  You’ll need to provide a simple text file with the samples IDs you’d like to keep arranged in a single column (in the example below, we have called this ‘sample_id_fp’).</p>

<div class="highlighter-rouge"><pre class="highlight"><code>filter_fasta.py -i MiSeq12345678_seqs.fna --sample_id_fp MappingFile.txt -o Project.fna
</code></pre>
</div>

<h2 id="generate-otu-table">Generate OTU table</h2>

<p>At this point, you’ll have to decide if you want to pick OTU’s <em>de novo</em>, use a closed reference method, or use an open reference method.  You can read more about these methods, and their respective advantages/disadvantages <a href="">here</a>. Don’t be deceived by this short line of code.  There’s a lot of important stuff that’s happening here, so you may want to read up a bit on this.</p>

<h3 id="option-1">Option 1:</h3>

<div class="highlighter-rouge"><pre class="highlight"><code>pick_de_novo_otus.py -i Project.fna -o DeNovoOTUs 
</code></pre>
</div>

<h3 id="option-2-longer-way-but-easier-to-extract-the-chimeras">Option 2: (longer way, but easier to extract the chimeras)</h3>

<p>pick_otus.py -i Project.fna –enable_rev_strand_match –otu_picking_method cdhit -s 0.97 –prefix_prefilter_length 200 -o otus</p>

<p>pick_rep_set.py -f Project.fna -i otus/Project_otus.txt –rep_set_picking_method most_abundant -o rep_set.fna</p>

<p>align_seqs.py -i rep_set.fna -t /home/Reference/core_set_aligned.fasta.imputed.txt -o aligned_seqs</p>

<p>assign_taxonomy.py -i rep_set.fna -o assigned_taxonomy –confidence 0.8</p>

<h2 id="remove-pcr-chimeras">Remove PCR chimeras</h2>

<p><a href="http://genome.cshlp.org/cgi/pmidlookup?view=long&amp;pmid=21212162">chimeras</a> are PCR amplicons formed from two separate parent DNA molecules during the PCR reaction.  These need to be removed, otherwise they would be interpreted as novel organisms and would therefore erroneously inflate community diveristy. The output file is a txt file that has the list of sequence numbers.</p>

<div class="highlighter-rouge"><pre class="highlight"><code>identify_chimeric_seqs.py -i aligned_seqs/rep_set_aligned.fasta -m ChimeraSlayer -o chimeric_seqs.txt -a /home/misic/Reference/core_set_aligned.fasta.imputed.txt
</code></pre>
</div>

<h2 id="filtering-and-subsetting">Filtering and subsetting</h2>

<p>Produce a .biom file from your data.  You can read about the .biom file format <a href="http://biom-format.org/">here</a></p>

<div class="highlighter-rouge"><pre class="highlight"><code>make_otu_table.py -i otus/Project_otus.txt -t assigned_taxonomy/rep_set_tax_assignments.txt -o otu_table.biom -e chimeric_seqs.txt
</code></pre>
</div>

<p>Remove OTUs from the table that only have a single representative sequence, a.k.a. ‘singletons’, as well as OTUs with fewer than 10 sequences</p>

<div class="highlighter-rouge"><pre class="highlight"><code>filter_otus_from_otu_table.py -i otu_table.biom -n 10 -o otu_table2.biom
</code></pre>
</div>

<p>If you want to see the most (or least) abundant OTUs in your table</p>

<div class="highlighter-rouge"><pre class="highlight"><code>filter_otus_from_otu_table.py -i otu_table_2.biom --min_count_fraction 0.01 -o otu_table_onlygreaterthan1percent.biom
</code></pre>
</div>

<p>We use file with the otu names of the mitochondria, cyanobacteria, and unassigned taxa and remove those from the OTU table.  Depending on your project, you may or may not want to remove these. For work involving stool, I remove cyanobacteria. However, if you are performing an enviornmental study, you may choose to keep these in.</p>

<div class="highlighter-rouge"><pre class="highlight"><code>$ filter_otus_from_otu_table.py -i otu_table2.biom -e Mito_Cyano_and_unassigned.txt -o otu_table3.biom
</code></pre>
</div>

<p>Now subset your data to select only those columns of the taxa table the correspond to the samples you’re interested in analyzing</p>

<div class="highlighter-rouge"><pre class="highlight"><code>filter_samples_from_otu_table.py -i otu_table_3.biom --sample_id_fp MappingFile_SoilOnly.txt -o otu_table_4SoilOnly.biom
</code></pre>
</div>

<p>Summarize your .biom file to produce a simple text file with taxa</p>

<div class="highlighter-rouge"><pre class="highlight"><code>biom summarize-table -i otu_table3.biom -o Table3Summary.txt
</code></pre>
</div>

<h2 id="rarify-to-desired-depth">rarify to desired depth</h2>

<div class="highlighter-rouge"><pre class="highlight"><code>single_rarefaction.py -i otu_table_4soilonly.biom -d 2900 -o otu_table_5_soil_2900.biom
</code></pre>
</div>

<h2 id="create-rel-abund-for-taxa">create rel abund for taxa</h2>

<div class="highlighter-rouge"><pre class="highlight"><code>summarize_taxa_through_plots.py -m /home/misic/MAD/Files/MappingFile_MAD_v3.txt -c Treatment -o TaxaSumTreatment -i otu_table_3.biom 
</code></pre>
</div>

<h2 id="get-counts-for-taxa">Get counts for taxa</h2>

<div class="highlighter-rouge"><pre class="highlight"><code>summarize_taxa.py -i otu_table_3.biom -a -o TaxaSumAbs -m /home/misic/MAD/Files/MappingFile_MAD_v3.txt 
# Flag: -L 2 phylum ; -L 3 class ; -L 4 order ; -L 5 family -L 6 genus ; -L 7 species
example: summarize_taxa_through_plots.py -m /home/misic/MAD/Files/MappingFile_MAD_v3.txt -c Treatment -o TaxaSumTreatment -i otu_table_3.biom  -L 5
</code></pre>
</div>

<h2 id="group-samples-by-treatment-group">Group samples by treatment group</h2>

<div class="highlighter-rouge"><pre class="highlight"><code>filter_samples_from_otu_table.py -i otu_table_3.biom --sample_id_fp MappingFile_MAD_SoilOnly.txt -o otu_table_4soilonly.biom
</code></pre>
</div>

<h2 id="generate-beta-diversity-plots">Generate beta diversity plots</h2>

<div class="highlighter-rouge"><pre class="highlight"><code>beta_diversity_through_plots.py -i otu_table_5_soil_2900.biom -m MappingFile_MAD_SoilOnly.txt -o BetaDiv_Soil -t rep_set.tre 
</code></pre>
</div>

<h2 id="filter-taxa-based-on-rel-abund">filter taxa based on rel abund</h2>

<div class="highlighter-rouge"><pre class="highlight"><code>$ filter_otus_from_otu_table.py -i otu_table_3.biom --min_count_fraction 0.01 -o otu_table_onlygreaterthan1percent.biom
</code></pre>
</div>

<h2 id="relative-abundance-and-plots">relative abundance and plots</h2>

<div class="highlighter-rouge"><pre class="highlight"><code>summarize_taxa_through_plots.py -m MappingFile.txt -c Treatment -o TaxaSumTreatment -i otu_table_3.biom 
</code></pre>
</div>

<p>Optional flags include for this function include: 
* <code class="highlighter-rouge">-L 2</code> for phylum 
* <code class="highlighter-rouge">-L 3</code> for class
* <code class="highlighter-rouge">-L 4</code> for order
* <code class="highlighter-rouge">-L 5</code> for family
* <code class="highlighter-rouge">-L 6</code> for genus
* <code class="highlighter-rouge">-L 7</code> for species</p>

<p>Example: summarize_taxa_through_plots.py -m MappingFile.txt -c Treatment -o TaxaSumTreatment -i otu_table_3.biom -L 5.  Only gives data at family level. Groups samples into treatment groups</p>

<p>Produce a table with absolute sequence counts.</p>

<div class="highlighter-rouge"><pre class="highlight"><code>summarize_taxa.py -i otu_table_3.biom -a -o TaxaSumAbs -m MappingFile.txt 
</code></pre>
</div>

<p>Identify the core microbiome in your samples at different taxanomic levels (for example, which OTUs are found in 90% of the samples).</p>

<div class="highlighter-rouge"><pre class="highlight"><code>compute_core_microbiome.py -i otu_table.biom -o otu_table_core_fast --mapping_fp MappingFile.txt --valid_states "Treatment:Fast"
</code></pre>
</div>

<h2 id="alpha-diversity">Alpha diversity</h2>

<p>Obtain the alpha diversity values for the listed at 4 phylogenetic levels</p>

<p>There are many options for alpha diversity metrics, but the ones we use most commonly include:</p>

<ul>
  <li>observed_species</li>
  <li>shannon</li>
  <li>simpson</li>
  <li>simpson_e</li>
  <li>PD_whole_tree</li>
  <li>chao1</li>
  <li>fisher_alpha</li>
</ul>

<div class="highlighter-rouge"><pre class="highlight"><code>alpha_diversity.py -m shannon,chao1,PD_whole_tree,observed_species -i otu_table.biom -o AlphaDiv.txt -t rep_set_aligned.tre 
</code></pre>
</div>

<h2 id="rarefaction">rarefaction</h2>

<p>Generate ‘collectors curves’ (A.K.A. rarefaction plots) that can be visualized as HTML</p>

<div class="highlighter-rouge"><pre class="highlight"><code>alpha_rarefaction.py -O 10 -a -i otu_table3.biom -o alpha_rare_curves -m MappingFile.txt -t rep_set_aligned.tre -p parameters.txt
</code></pre>
</div>

<p>rarify to desired depth, in this example, 2900 sequences</p>

<div class="highlighter-rouge"><pre class="highlight"><code>single_rarefaction.py -i otu_table.biom -d 2900 -o otu_table_2900.biom
</code></pre>
</div>

<h2 id="optional-qiime-core-diversity-analysis">Optional: QIIME ‘core diversity analysis’</h2>

<p>If so desired, you can use the function below to carry out several QIIME diversity analyses together.  This is a basic workflow that begins with a BIOM table, mapping file, and optional phylogenetic tree.</p>

<div class="highlighter-rouge"><pre class="highlight"><code>core_diversity_analyses.py -i otu_table.biom -m MappingFile.txt -o CoreAnalyses -t rep_set_aligned.tre -c Timepoint,Exper_name,Breed_Strain -e 5000
</code></pre>
</div>

<p>The included scripts are those run by the workflow scripts alpha_rarefaction.py, beta_diversity_through_plots.py, summarize_taxa_through_plots.py, plus the (non-workflow) scripts make_distance_boxplots.py, compare_alpha_diversity.py, and otu_category_significance.py. To update parameters to the workflow scripts, you should pass the same parameters file that you would pass if calling the workflow script directly.</p>

<p>Additionally, a table summary is generated by running the ‘biom summarize-table’ command (part of the biom-format package). To update parameters to this command, your parameters file should use ‘biom-summarize-table’ (without quotes) as the script name. See http://qiime.org/documentation/qiime_parameters_files.html for more details.</p>

<h2 id="beta-diversity">Beta diversity</h2>

<p>This is to create the distance matrix which is then used for the next script, compare categories</p>

<div class="highlighter-rouge"><pre class="highlight"><code>beta_diversity_through_plots.py -i otu_table.biom -m MappingFile.txt -o BetaDiv -t rep_set.tre -p parameters.txt
</code></pre>
</div>

<div class="highlighter-rouge"><pre class="highlight"><code>beta_diversity.py -i otu_table.biom -m weighted_unifrac,unweighted_unifrac,bray_curtis,binary_jaccard -t rep_set_aligned.tre -o BetaDivMatricesV10
</code></pre>
</div>

<p>With Beta diversity you also have many choices for metrics to you.  We most often look at:
* weighted_unifrac
* unweighted_unifrac
* bray_curtis
* binary_jaccard</p>

<p>This script uses anosim (Analysis of similarities) to analyze if groups vary by timepoint. Output is an R value (from -1 to 1) and a p value. The more a category influences microbiota, the larger the R values will be, and the smaller the p value will be.</p>

<div class="highlighter-rouge"><pre class="highlight"><code>compare_categories.py -i otu_table.biom -m MappingFile.txt -i BetaDivMatrices/weighted_unifrac_otu_table.txt -c Timepoint --method anosim -o WU_Timepoint
</code></pre>
</div>

<p>Note: you must explicitly state which statisical test you would like to use to compare categories.  You options include:
Also Note: you input biom. file could be rarefied (or not), or filtered to only be certain groups</p>

<ul>
  <li>adonis</li>
  <li>anosim</li>
  <li>best</li>
  <li>morans_i</li>
  <li>mrpp</li>
  <li>permanova</li>
  <li>permdisp</li>
  <li>dbrda</li>
</ul>

<p>You may also find it useful to make boxplots showing distances.  You can do this as follows:</p>

<div class="highlighter-rouge"><pre class="highlight"><code>make_distance_boxplots.py -d BetaDivMatrices/unweighted_unifrac_otu_table.txt -o DistanceHist -m MappingFile.txt -f Treatment
</code></pre>
</div>

<h2 id="machine-learning-approaches">Machine learning approaches</h2>

<div class="highlighter-rouge"><pre class="highlight"><code>supervised_learning.py -i otu_table.biom -m MappingFile.txt -c Treatment -o MachineLearningResults -v -e cv10
</code></pre>
</div>



    <div class="tags">
        
        <b>Tags: </b>
        
        
        
        <a href="tag_bioinformatics.html" class="btn btn-default navbar-btn cursorNorm" role="button">bioinformatics</a>
        
        
        
        <a href="tag_microbiome.html" class="btn btn-default navbar-btn cursorNorm" role="button">microbiome</a>
        
        
        
    </div>



    <div id="disqus_thread"></div>
    <script type="text/javascript">
        /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
        var disqus_shortname = 'http-hostmicrobe-org'; // required: replace example with your forum shortname

        /* * * DON'T EDIT BELOW THIS LINE * * */
        (function() {
            var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
            dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
            (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
        })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>


</div>

<hr class="shaded"/>

<footer>
            <div class="row">
                <div class="col-lg-12 footer">
               &copy;2020 PennVet Center for Host-Microbial Interactions. All rights reserved. <br />
 Site last generated: Mar 23, 2020 <br />
<p><img src="images/CHMI_logo.png" alt="Company logo"/></p>
                </div>
            </div>
</footer>


        </div>
    <!-- /.row -->
</div>
<!-- /.container -->
    </div>

</body>

<!-- the google_analytics_id gets auto inserted from the config file -->



<script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','//www.google-analytics.com/analytics.js','ga');ga('create','UA-131349832-1','auto');ga('require','displayfeatures');ga('send','pageview');</script>


</html>
